---
title: "Fig 3 Code"
output: html_document
editor_options: 
  chunk_output_type: inline
---
```{r}
library(tidyverse)
library(data.table)
library(cowplot)
library(wesanderson)
library(ggplot2)
library(MASS) 
library(qtl)
library(lme4) 
library(arm) 
library(lsmeans) 
library(pbkrtest)
library(cowplot) 
library(viridis)
library(qtlcharts)
library(readr)
library(ggfortify)
library(ggmap)
library(maps)
library(qtlcharts)
library(Hmisc)
library(dabestr)
```


```{r}

data <- read.csv("data/fig_3/metepec_lipids.csv")

#estimate breeding values

traits <- colnames(data)
#traits 72 to 140 hold LIPID values

for (i in 3:57) {
#fit mixed model taking into account the three replicate blocks in each of the fields
mix = lmer(data[,i] ~ MUM_GEN + (1|BLOCK_FIELD),data=data, na.action=na.exclude)
#sjp.lmer(mix, type="re") #plots the random effects as deviations from the main effect 
#Extract Breeding Values
bv <- lsmeans(mix, specs = ~ MUM_GEN)
bv <- summary(bv) 
bv=bv[,1:2]
bv[,2]=round(bv[,2])
colnames(bv)[1]="RIL"
colnames(bv)[2]=traits[i]

}

#calculate average values per RIL

data_avg <- data %>%
  group_by(data$RILS) %>%
  summarise_at(vars(-RILS), funs(mean(., na.rm=TRUE)))

write_csv(data_avg,"data/fig_3/data_avg.csv")
#read cross object


cross <- read.cross(format="csv",file="data/fig_3/cross_object_metepec.csv", genotypes=c("AA","AB","BB"), alleles=c("B73","PT"), crosstype = "riself")

summary(cross)
plotMap(cross)
head(cross$pheno)

data_cross <- read_csv("data/fig_3/cross_object_metepec.csv")

data_cross <- data_cross[3:54,]

geno.image(cross)


```



```{r}
cross <- calc.genoprob(cross, step=1)

#We now perform single QTL analysis for each trait using Haley-Knott regression

#LPCs QTL

for (i in 3:12){
trait=colnames(cross$pheno)[i]
out.hk <- scanone(cross,pheno.col=i,method="hk")
plot(out.hk, col=c("black"), lwd = 0.7, main=trait) #all plots on same scale
#Estimate LOD cut-off using permutation; add 5% cut off to plots
perm <- scanone(cross, pheno.col=i, n.perm=1000, method="hk")
cut.off <- summary(perm, alpha=0.05)[1]
abline(h=cut.off,col="red",lty=2) #5% cut off for hk
}

#find markers 


lpc_markerchr3 <- bayesint(out.hk, 3, 0.95, expandtomarkers=TRUE)
lpc_markerchr7 <- bayesint(out.hk, 2, 0.95, expandtomarkers=TRUE)

find.marker(cross, chr=2, pos= 7)
find.marker(cross, chr=7, pos= 124)

#plot QTLs

out.hk %>%
  filter(chr == 4) %>%
  ggplot(aes(x = pos, y = lod)) +
  geom_line(aes(color = "blue")) +
  geom_hline(yintercept = cut.off,col="red",lty=2)  +
  ylim(0, 15) +
  geom_rug(sides="b")

#Effect plots

effect_plot_lpc_chr3 <- data_cross %>%
  #filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = LPCs/10E6, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot() +
  ylab("LPCs") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qLPCs3@8.5") +
  theme(legend.position = "none")


effect_plot_lpc_chr7 <- data_cross %>%
  #filter(S7_161685599 != "NA") %>%
  ggplot(aes(x = S7_161685599, y = LPCs/10E6, colour = S7_161685599)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot() +
  ylab("LPCs") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qLPCs7@162") +
  theme(legend.position = "none")


#epistasis plots LPCs

epistasis_chr3_B73_LPCs <- data_cross %>%
  filter(S3_8542287 == "AA" & S7_161685599 == "AA" | S7_161685599 == "BB") %>%
  ggplot(aes(x = S7_161685599, y =  LPCs/10E6, colour = S7_161685599)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot() +
  ylab("LPCs") +
  theme(axis.title.x=element_blank()) +
  ggtitle("3@8.54Mb_B73") +
  theme(legend.position = "none")

epistasis_chr3_PT_LPCs <- data_cross %>%
  filter(S3_8542287 == "BB" & S7_161685599 == "AA" | S7_161685599 == "BB") %>%
  ggplot(aes(x = S7_161685599, y =  LPCs/10E6, colour = S7_161685599)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot() +
  ylab("LPCs") +
  theme(axis.title.x=element_blank()) +
  ggtitle("3@8.54Mb_PT") +
  theme(legend.position = "none")


#Effect Plots PCs

PCs <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PCs/10E6, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PCs") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPCs3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

PCs_LPCs <- data_cross %>%
  #filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PCs_LPCs, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot() +
  ylab("PCs/LPCs") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPCs/LPCs3@8.5") +
  theme(legend.position = "none")

A1 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC32_1.LPC_18_1, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC32:1/LPC18:1") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC32:1/LPC18:1-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))
        

A2 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC34_1.LPC_18_1, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC34:1/LPC18:1") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC34:1/LPC18:1-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

A3 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC36_1.LPC_18_1, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC36:1/LPC18:1") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC36:1/LPC18:1-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

A4 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC36_2.LPC_18_1, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC36:1/LPC18:1") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC36:1/LPC18:1-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

A5 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC32_1.LPC_18_2, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC32:1/LPC18:2") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC32:1/LPC18:2-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

A6 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC34_1.LPC_18_2, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC34:1/LPC18:2") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC34:1/LPC18:2-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

B1 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC36_1.LPC_18_2, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC36:1/LPC18:2") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC36:1/LPC18:2-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

B2 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC36_2.LPC_18_2, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC36:2/LPC18:2") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC36:2/LPC18:2-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

B3 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC_34_4/LPC_18_1, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC34:4/LPC18:1") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC34:4/LPC18:1-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

B4 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC_34_4/LPC_18_2, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC34:4/LPC18:2") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC34:4/LPC18:2-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

B5 <- data_cross %>%
  filter(S5_204089927 != "NA") %>%
  ggplot(aes(x = S5_204089927, y = PC_34_1/10E5, colour = S5_204089927)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC34:1") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC34:1-5@204") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

B6 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC_34_2/10E5, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC34:2/10E5") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC34:2-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

C1 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC_34_3/10E5, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC34:3/10E5") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC34:3-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

C2 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC_34_4/10E3, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC34:4/10E3") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC34:4-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

C3 <- data_cross %>%
  filter(S5_204089927 != "NA") %>%
  ggplot(aes(x = S5_204089927, y = PC36_1/10E4, colour = S5_204089927)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC36:1") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC36:1-5@204") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

C4 <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  ggplot(aes(x = S3_8542287, y = PC_36_5_A/10E5, colour = S3_8542287)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(11) +
  ylab("PC36:5A/10E5") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC36:5A-3@8.5") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

C5 <- data_cross %>%
  filter(S2_2677779 != "NA") %>%
  ggplot(aes(x = S2_2677779, y = PC_36_4_A/10E5, colour = S2_2677779)) +
  geom_jitter(position=position_jitter(0.1), size = 2) +
  #ylim(0,2e+05) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  theme(legend.position="none") +
  theme_cowplot(9) +
  ylab("PC36:4A/10E5") +
  theme(axis.title.x=element_blank()) +
  ggtitle("qPC36:4A-2@2.6") +
  theme(legend.position = "none") +
  theme(plot.title = element_text(size=11))

plot_grid(A1, A2, A3, A4, A5, A6, B1, B2, B3, B4, B5, B6, C1, C2, C3, C4, C5, PCs, ncol=6)

dabest_try <- data_cross %>%
  filter(S3_8542287 != "NA") %>%
  dabest(S3_8542287, LPCs, 
         idx = c("AA", "BB"),
         paired = FALSE)

plot(dabest_try)
 

```

Expression analysis of ZmPla1.2

```{r}
exp_values <- read_csv("data/fig_3/ZmPLA1_exp.csv")

exp_values %>% 
	filter(tissue_key=="0") %>%
  #filter(line=="B73"|line=="PT") %>%
  #filter(fertilizer_tx=="high") %>%
	ggplot(aes(x = temp_tx, y = ZmPla1.2, color = temp_tx)) +
  scale_color_manual(values=c("#1260ad","#df1b23", "#fd6820", "#14ade8")) +
	geom_jitter(position=position_jitter(0.1), size = 2) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", 
               color = "black") +
  #ylim(0,8) +
  ylab("ZmPla1.2 expression levels") +
  xlab("Temperature conditions") +
  theme_cowplot(13) +
  facet_grid(cols = vars(line), rows = vars(fertilizer_tx)) +
  #coord_fixed(ratio = 0.9) +
  theme(legend.position="none") 
```

Haplotype Network plot

```
library(dplyr)
library(vcfR)
library(pegas)


traits <- read.table("/Volumes/GoogleDrive/My\ Drive/repos/soilP/inst/notebooks/P_traits.txt", header = TRUE)

extdata <-"/Volumes/GoogleDrive/My\ Drive/repos/soilP/inst/extdata"

accn_file <- paste(extdata,"germinate_SeeD_GWAS_GBS_4022.tab", sep = "/")

accn_info <- read.table(file = accn_file, header = TRUE, quote = "",sep ="\t")

SEED_GWAS_GID <- read.table(file = file.path(extdata, "SEED_GWAS_GID.tab"),
                            header = TRUE, quote = "",sep ="\t")

# There are one or more GWAS seed identifier per general identifier in the table

GWAS_ID <- read.table(file = paste(extdata, "GWAS_ID.tab", sep = "/"), quote = "",sep ="\t")
GWAS_ID$taxa <- paste(GWAS_ID$V1,GWAS_ID$V2,GWAS_ID$V3,GWAS_ID$V4, sep = ":")

colnames(GWAS_ID)[1] <-"sample_id"

GWAS_ID <- GWAS_ID %>% 
  dplyr::select(sample_id,taxa) %>%
  inner_join(SEED_GWAS_GID)

GWAS_ID %>% 
  inner_join(accn_info,by = c(accn_GID = "general_identifier")) %>%
  #dplyr::filter(countries_country_code3=="MEX") %>%
  dplyr::select(taxa,country="countries_country_code3", collnumb)

mex <- accn_info %>%
  dplyr::filter(!is.na(locations_longitude) & !is.na(locations_longitude)) %>%
  dplyr::filter(countries_country_code3 =="MEX") %>%
  dplyr::select(accn_GID  ="general_identifier",collnumb, lon ="locations_longitude",
                lat="locations_latitude", country = "countries_country_code3") 

assoc <- mex %>% 
  inner_join(GWAS_ID) %>% dplyr::select(taxa,collnumb) %>%
  inner_join(traits) %>% 
  arrange(locations_elevation) %>%
  group_by(locations_elevation)

# Select just Mexican accessions genotype or Teosintes not georeferenced

vcf_file <-  "/Users/fvrodriguez/Projects/NCSU/05_PLA1/zmpla12_merged_het0_AGPv3_phased.vcf"

vcf <- vcfR::read.vcfR(vcf_file) 
ncol(vcf@gt) - 1

is_mexican <- colnames(vcf@gt) %in% c("FORMAT",assoc$taxa) 
is_teosinte <- grepl("TIL.*|Teo",colnames(vcf@gt), perl = TRUE)

vcf <-vcf[,is_mexican | is_teosinte ]
ncol(vcf@gt) - 1

# 916 mexican  out of  1379 initial
dna <- vcfR2DNAbin(vcf)
as.alignment(dna)
h <- haplotype(dna, what="label")


haptab <- stack(setNames(attr(h, "index"), rownames(h))) %>%
  dplyr::mutate(h = as.numeric(ind)) %>%
  dplyr::select(h, label =ind, dnaid= values)

sumhap <- haptab %>% 
  dplyr::group_by(h,label) %>%
  dplyr::summarise(count= length(h))  %>%
  dplyr::arrange(-count) %>% ungroup() %>%
  dplyr::mutate(countrank = row_number()) %>%
  dplyr::mutate(cumcount = cumsum(count)) %>% 
  dplyr::mutate(cumfreq  = cumcount/sum(count)) %>% 
  dplyr::mutate(is_major = cumfreq < 0.9) %>% 
  as.data.frame()

majorhid <- sumhap$h[sumhap$is_major]
majordnaid  <- haptab$dnaid[haptab$h %in% majorhid ]

majordna <- dna[majordnaid,]
nrow(dna)/2
nrow(majordna)/2


h <- haplotype(majordna)

check_mexicana <- function(x,h,dna) {
  h_names <- rownames(dna)[attr(h,"index")[[x]]]
  if(any(grepl("TIL08|TIL25",h_names))) {x} else {NULL}
}


hapind <- stack(setNames(attr(h, "index"), rownames(h))) %>% 
  dplyr::mutate(h = as.numeric(ind)) %>%
  dplyr::select(h, label =ind, dnaid = values) %>%
  dplyr::mutate(seqname = rownames(majordna)[dnaid]) %>%
  dplyr::mutate(seq = as.alignment(majordna)$seq) %>%
  dplyr::mutate(taxa = gsub("_.*","",seqname,perl = TRUE)) %>%
  dplyr::left_join(assoc) %>%
  dplyr::mutate(is_mexicana    = grepl("TIL08|TIL25", taxa, perl = TRUE)) %>%
  dplyr::mutate(is_parviglumis = grepl("TIL|Teo", taxa, perl = TRUE) & !is_mexicana) 

table(cut(hapind$locations_elevation, breaks = c(0,1000,2000,4000)), right = TRUE)
  
hapind$ele_class <- cut(
  hapind$locations_elevation, breaks = c(0,1000,2000,4000), 
  labels = c("low","mid","high"))


hapind$ele_bin <- cut(
  hapind$locations_elevation, breaks = seq(0, 3100, by= 100), 
  labels = seq(50, 3100, by= 100)) %>% 
  as.character() %>%
  as.integer()

hap_ele <- hapind %>% dplyr::group_by(ele_bin) %>% dplyr::select(taxa, label) %>%
  dplyr::summarise(
                II = sum(label=="II")/length(taxa),
                III = sum(label=="III")/length(taxa),
                VI = sum(label=="VI")/length(taxa),
                n = length(taxa)) %>% 
                 dplyr::rename( Elevation ="ele_bin") %>%
                dplyr::filter( n >10) %>% 
                tidyr::pivot_longer(cols = c("II","III","VI"),
                                             values_to = "Frequency", 
                                             names_to = "Haplotype")

quartz()
hap_ele %>% dplyr::filter(Haplotype == "VI") %>%
  ggpubr::ggscatter(  x = "Elevation", y = "Frequency",color = "#88CCEE", add = "loess", conf.int = TRUE) +
  ggplot2::ggtitle("Haplotype VI") +
  ggplot2::theme(legend.title = ggplot2::element_blank()) 


high_limit <- 2000
low_limit <- 1000

 hapind %>% 
  dplyr::filter(locations_elevation >= high_limit | locations_elevation <= low_limit ) %>%
  dplyr::select(h:label, is_mexicana) %>% nrow()

 hapind %>% 
   dplyr::filter(is_mexicana)
 
 hapind %>% 
   dplyr::filter(is_parviglumis)


hapind$pop <- hapind$ele_class 
hapind$pop <- factor(hapind$pop, levels = c("low","mid","high"))

# hapind$pop[hapind$is_parviglumis] <- "parviglumis"    
# hapind$pop[hapind$is_mexicana] <- "mexicana"  

# hapind$pop <- factor(hapind$pop, 
#                      levels = c("mexicana","parviglumis","low","high"))
#pal <- c("#FFFFFF","#DDDDDD","#CC6677","#88CCEE")

pie_tab <- with(hapind, table(hap=label, pop=pop))

net <- pegas::haploNet(h)


pal <- c("#CC6677","#BCDCAD", "#88CCEE")



p <- plot(net, legend = FALSE,
     size=attr(net, "freq")/50, 
     scale.ratio = 2, cex = 0., 
     pie= pie_tab,
     col = "white",
     bg=pal,
     show.mutation = 2,threshold =0 )

pdf(file="hapnetwork.pdf")

plot(net, legend = FALSE,
     labels = FALSE,
     size=attr(net, "freq")/50, 
     scale.ratio = 2, cex = 0., 
     pie= pie_tab,
     col = "white",
     bg=pal,
     show.mutation = 2,threshold =0, )
size.adjust <- c(0.8, 0.6,0.7, 1.5,1.5,0.6,1,1.2,1.5)
angle.adjust <- c(2*pi/3,2*pi/3,2*pi/3,-pi/2, pi/2, 2*pi/3, -pi/3, -pi/2, 0)
text(p$xx + cos(angle.adjust)*p$size*size.adjust , 
     p$yy + sin(angle.adjust)*p$size*size.adjust ,
     labels = p$labels, srt = 90)
legend(-25,30, box.lty = 0, colnames(pie_tab), fill=pal,  title = "Population by Elevation", ncol =3)
dev.off()

```