---
title: "Fig 3 Code"
output: html_document
editor_options:
  chunk_output_type: inline
---

# Annotated LOD profile

```{r}
library(dplyr)
library(qtl)
library(GenomicRanges)
library(ggbio)
library(igraph)
library(qtl)
source("detect_peaks.R")

##### QTL analysis ##########################################################
n_threads <- 5
n_perm <- 1000
seed <- 1234567890
set.seed(seed)
data_dir <- "../data/fig_3"
geno_file  <- file.path(data_dir, "B73xPT_AGPv3_genotype.csv")
pheno_file <- file.path(data_dir, "B73xPT_combined_phenotype.csv")

cross <- read.cross(
  format = "csvs",  file = geno_file,
  na.strings = c("-",NA),
  phefile = pheno_file,
  genotypes = c("A","B")) %>%
  convert2riself() %>%
  jittermap() %>%
  calc.genoprob( step = 1)

# Select phenotypes with up to 50% missing data ###############################
# missing phenotypes will generate errors

nind <- nind(cross)
nphe <- nphe(cross)

missing <- colSums(is.na(cross$pheno))
missing_thresh <- 0.5

pheno_col <- which( missing < missing_thresh * nind + 4) # this form comes from the readcross warning
                                                         # RILs with genotype but no phenotype
pheno_col <- pheno_col[-1]


single_scan <- scanone(cross, pheno.col = pheno_col, method = "hk")
perms <- scanone(cross, pheno.col = pheno_col,
                 n.perm = n_perm, method = "hk", n.cluster = n_threads)


peaks <- get_peak_table(perms = perms, single_scan = single_scan)

refined_peaks <- refine_peaks(peaks, perms = perms, single_scan = single_scan) %>%
  select(trait,lod,thresh_adj,thresh, everything()) %>%
  arrange(-lod) %>% tibble() %>%
  print(n = 150)

new_peaks  <- refined_peaks[refined_peaks$lod > refined_peaks$thresh_adj,]

# Variance explained by  3@8.5

pla12_pos <- pull.map(cross = cross)[[3]]["S3_8542287"]

q1 <- makeqtl(sim.geno(cross), 3, pla12_pos, "3@8.5")

qtraits <-  c("PCs","LPCs","PCs.LPCs")

peaks %>%
  dplyr::filter(trait %in% qtraits) %>%
  dplyr::select(trait, pos, marker, marker_lod)

trait_var <- lapply(qtraits, function(x){
  fit <- fitqtl(sim.geno(cross) , 
       pheno.col=which(colnames(cross$pheno) == x), 
       dropone = TRUE,
       q1)
  fit$result.full[1,5]
})

names(trait_var) <- qtraits
trait_var

##### Output Profile ##########################################################

p1 <- single_scan  %>%
  dplyr::select( chr,pos, LPCs, PCs,PCs.LPCs) %>%
  tidyr::pivot_longer(cols = c("LPCs", "PCs","PCs.LPCs"),
                      names_to = "trait",values_to = "LOD") %>%
  dplyr::mutate(trait = factor(trait, levels = c( "PCs","LPCs", "PCs.LPCs"))) %>%
  ggplot2::ggplot() +
  ggplot2::geom_line(aes(x = pos, y = LOD, col = trait))+
  ggplot2::xlab("Chromosome") +
  ggplot2::ylab("LOD") +
  ggplot2::geom_hline(aes(yintercept = LPCs_thresh), col = pal[1], lty =2)+
  ggplot2::facet_wrap(.~ as.factor(chr) ,
                      scales = "free_x",
                      strip.position = "bottom",
                      ncol = 10)  +
  ggplot2::geom_rug(aes(x = pos),sides="b", size =0.01) +
  ggpubr::theme_classic2(base_size = 18) +
  ggplot2::theme(legend.position = "none",
                 strip.background = ggplot2::element_blank(),
                  strip.placement = "outside",
                  axis.text.x = ggplot2::element_text(size =8, angle = 90))

p2 <- single_scan  %>%
  dplyr::filter(chr==3) %>%
  dplyr::select( chr,pos, LPCs, PCs,PCs.LPCs) %>%
  tidyr::pivot_longer(cols = c("LPCs", "PCs","PCs.LPCs"),
                      names_to = "trait",values_to = "LOD") %>%
  dplyr::mutate(trait = factor(trait, levels = c( "PCs","LPCs", "PCs.LPCs"))) %>%
  ggplot2::ggplot() +
  ggplot2::geom_line(aes(x = pos, y = LOD, col = trait))+
  ggplot2::xlab("Position cM") +
  ggplot2::ylab("") +
  ggplot2::geom_hline(aes(yintercept = LPCs_thresh), col = pal[1], lty =2)+
  ggplot2::geom_vline(aes(xintercept = pla12_pos), col = "red") +
  ggplot2::geom_rug(aes(x = pos),sides="b", size =0.1) +
  ggpubr::theme_classic2() +
  ggplot2::theme( plot.margin = unit(c(0,0,0,0), "lines"),
                  legend.position = "none")

plot.with.inset <-
  cowplot::ggdraw() +
  cowplot::draw_plot(p1) +
  cowplot::draw_plot(p2, x = 0.3, y = .45, width = .68, height = .45)


png(file = "Figure3A.png", height = 616, width = 1168)

# quartz(height = 2*3.08, width = 2*5.84)

print(plot.with.inset)
 
grid::grid.text(expression(italic("ZmPla1.2")),
                just = "left", x = 0.475, y = 0.92)
grid::grid.text(expression(italic("qPCs/LPCs")),
                just = "left", x = 0.525, y = 0.85)
grid::grid.text(expression(italic("qLPCs")),
                just = "left",  x = 0.54, y = 0.675)
grid::grid.text(expression(italic("qPCs")),
                just = "left", x = 0.55, y = 0.63)

grid::grid.lines(x = unit(c(0.512, 0.523), "npc"),
                 y = unit(c(0.85, 0.85), "npc"),
                 gp = grid::gpar(col= pal[3]))

grid::grid.lines(x = unit(c(0.51, 0.538), "npc"),
                 y = unit(c(0.656, 0.675), "npc"),
                 gp = grid::gpar(col= pal[2]))

grid::grid.lines(x = unit(c(0.515, 0.548), "npc"),
                 y = unit(c(0.60, 0.63), "npc"),
                 gp = grid::gpar(col= pal[1]))

grid::grid.lines(x = unit(c(0.255, 0.36), "npc"),
                 y = unit(c(0.215, 0.49), "npc"),
                 gp = grid::gpar(fill="black"),
                 arrow = grid::arrow(length = unit(0.1, "inches"),
                                     ends="last", type="closed"))
grid::grid.lines(x = unit(c(0.33, 0.95), "npc"),
                 y = unit(c(0.215, 0.49), "npc"),
                 gp = grid::gpar(fill="black"),
                 arrow = grid::arrow(length = unit(0.1, "inches"),
                                     ends="last", type="closed"))
dev.off()
```

## Effect Sizes.

```{r}
base_size <- 13

metabolite <- read.csv("pla_phyt_20.csv") %>%
  dplyr::mutate(geno = gsub("hom","mut", geno)) %>%
  dplyr::mutate(geno = gsub("wt","wt", geno)) %>%
  dplyr::rename(genotype = geno)


metabolite$PC_LPC <- metabolite$PCs/metabolite$LPCs
```

## Normalized per species

```{r}
pct_thresh <- 100

PCs_paired <-  PCs %>%
  dplyr::group_by(genotype,species) %>%
  dplyr::summarise(signal = mean(signal, na.rm =TRUE), signalZ = mean(signalZ, na.rm =TRUE)) %>%
  dplyr::ungroup() %>%
  dplyr::group_by(genotype) %>%
  dplyr::filter(!is.na(signal)) %>%
  dplyr::mutate(pct= prop.table(signal)*100) %>%
  dplyr::arrange(-pct) %>%
  dplyr::mutate(cum_pct = cumsum(pct)) %>%
  dplyr::filter(cum_pct <=  pct_thresh) %>%
  dplyr::ungroup() %>%
  dplyr::group_by(species) %>%
  dplyr::mutate( n = length(genotype)) %>%
  dplyr::filter( n > 1)

PCs_paired$label <- ""
PCs_paired$empty_label <- ""
PCs_paired$label[PCs_paired$genotype == "wt"] <- PCs_paired$species[PCs_paired$genotype == "mut"]
PCs_paired$empty_label[PCs_paired$genotype == "wt"] <- " "


PC_wilcox <- PCs_paired %>%
  dplyr::ungroup() %>%
  rstatix::wilcox_test(signalZ ~ genotype, paired =TRUE, exact = TRUE, alternative = "less")  %>%
  rstatix::add_xy_position(fun = "max", x = "geno")



LPCs_paired <- LPCs %>%
  dplyr::group_by(genotype,species) %>%
  dplyr::summarise(signal = mean(signal, na.rm =TRUE), signalZ = mean(signalZ, na.rm =TRUE)) %>%
  dplyr::ungroup() %>%
  dplyr::group_by(genotype) %>%
  dplyr::filter(!is.na(signal)) %>%
  dplyr::mutate(pct= prop.table(signal)*100) %>%
  dplyr::arrange(-pct) %>%
  dplyr::mutate(cum_pct = cumsum(pct)) %>%
  dplyr::filter(cum_pct <=  pct_thresh) %>%
  dplyr::ungroup() %>%
  dplyr::group_by(species) %>%
  dplyr::mutate( n = length(genotype)) %>%
  dplyr::filter( n > 1)


LPCs_paired$label <- ""
LPCs_paired$empty_label <- ""
LPCs_paired$label[LPCs_paired$genotype == "wt"] <- LPCs_paired$species[LPCs_paired$genotype == "mut"]
LPCs_paired$empty_label[LPCs_paired$genotype == "wt"] <- " "

LPC_wilcox <- LPCs_paired %>%
  dplyr::ungroup() %>%
  rstatix::wilcox_test(signalZ ~ genotype, paired =TRUE, exact = TRUE, alternative = "greater")  %>%
  rstatix::add_xy_position(fun = "max", x = "geno")
```

# Now with the RILs 
```{r}
cross <- read.cross(
  format = "csvs",  file = "B73xPT_AGPv3_genotype.csv",
  na.strings = c("-",NA),
  phefile = "B73xPT_combined_phenotype.csv",
  genotypes = c("A","B")) %>%
  convert2riself() %>%
  jittermap() %>%
  calc.genoprob( step = 1)

pheno <- read.csv("B73xPT_combined_phenotype.csv") %>%
    dplyr::group_by(id)

cross$pheno$id

S3_8542287 <-factor(pull.geno(cross,3)[, "S3_8542287"])
levels(S3_8542287) <-  c("B73", "PT")
to_plot <- dplyr::inner_join(
  data.frame( id = cross$pheno$id,
              genotype = S3_8542287),
  pheno
)


p031 <- metabolite %>%
  dplyr::filter(genotype %in% c("mut","wt")) %>%
  dplyr::mutate(genotype = factor(genotype, levels =  c("wt","mut"))) %>%
  ggplot2::ggplot(aes(x = genotype, y = PCs %>% scale() %>% as.vector(), colour = genotype)) +
  ggplot2::geom_jitter(position=position_jitter(0.1), size = 2) +
  ggplot2::stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  # ggplot2::ggtitle("B104 CRISPR") +
  ggplot2::ylab("PCs") +
  ggplot2::xlab("Genotype") +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad")) +
    ggpubr::theme_pubr(legend ="none", base_size=base_size) +
  ggplot2::theme(axis.title.x=element_blank())

p131 <- metabolite %>%
  dplyr::filter(genotype %in% c("mut","wt")) %>%
  dplyr::mutate(genotype = factor(genotype, levels =  c("wt","mut"))) %>%
  ggplot2::ggplot(aes(x = genotype, y = LPCs  %>% scale() %>% as.vector(), colour = genotype)) +
  ggplot2::geom_jitter(position=position_jitter(0.1), size = 2) +
  ggplot2::stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  # ggplot2::ggtitle("B104 CRISPR") +
  ggplot2::ylab("LPCs") +
  ggplot2::xlab("Genotype") +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad"))+
    ggpubr::theme_pubr(legend ="none", base_size=base_size) +
  ggplot2::theme(axis.title.x=element_blank())

p032 <- to_plot %>%
  dplyr::filter(!is.na(genotype)) %>%
  ggplot2::ggplot(aes(x = genotype, y = PCs %>% scale() %>% as.vector(), colour = genotype)) +
  ggplot2::geom_jitter(position=position_jitter(0.1), size = 2) +
  ggplot2::stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  ggplot2::ylab("PCs") +
  ggplot2::xlab("Genotype") +
  # ggplot2::ggtitle("B73 x PT RILs Chr03 @ 8542287") +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad"))+
    ggpubr::theme_pubr(legend ="none", base_size=base_size)+
  ggplot2::theme(axis.title.x=element_blank())


p132 <- to_plot %>%
  dplyr::filter(!is.na(genotype)) %>%
  ggplot2::ggplot(aes(x = genotype, y = LPCs %>% scale() %>% as.vector(), colour = genotype)) +
  ggplot2::geom_jitter(position=position_jitter(0.1), size = 2) +
  ggplot2::stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  ggplot2::ylab("LPCs") +
  ggplot2::xlab("Genotype") +
  # ggplot2::ggtitle("B73 x PT RILs Chr03 @ 8542287") +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad"))+
    ggpubr::theme_pubr(legend ="none", base_size=base_size) +
  ggplot2::theme(axis.title.x=element_blank())


p310 <- metabolite %>%
  dplyr::filter(genotype %in% c("mut","wt")) %>%
  dplyr::mutate(genotype = factor(genotype, levels =  c("wt","mut"))) %>%
  ggplot2::ggplot(aes(x = genotype, y = PC_LPC %>% scale() %>% as.vector(), colour = genotype)) +
  ggplot2::geom_jitter(position=position_jitter(0.1), size = 2) +
  ggplot2::stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  # ggplot2::ggtitle("B104 CRISPR") +
  ggplot2::ylab("PCs/LPCs") +
  ggplot2::xlab("Genotype") +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad"))+
    ggpubr::theme_pubr(legend ="none", base_size=base_size) +
  ggplot2::theme(axis.title.x=element_blank())

p320 <- to_plot %>%
  dplyr::filter(!is.na(genotype)) %>%
  ggplot2::ggplot(aes(x = genotype, y = PCs.LPCs %>% scale() %>% as.vector(), colour = genotype)) +
  ggplot2::geom_jitter(position=position_jitter(0.1), size = 2) +
  ggplot2::stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  ggplot2::ylab("PCs/LPCs") +
  ggplot2::xlab("Genotype") +
  # ggplot2::ggtitle("B73 x PT RILs Chr03 @ 8542287") +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad"))+
    ggpubr::theme_pubr(legend ="none", base_size=base_size) +
  ggplot2::theme(axis.title.x=element_blank())


PCs <- to_plot %>%
  dplyr::select(id, genotype, starts_with("PC_") & !matches("LPC") & !matches("\\.")) %>%
  dplyr::filter(!is.na(genotype)) %>%
  tidyr::pivot_longer(cols = starts_with("PC_"),
                      names_to = "species", values_to = "signal"
  ) %>%
  dplyr::group_by(species) %>%
  dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                log10 = log10(signal) %>% as.vector(),
                log10Z = scale(log10(signal)) %>% as.vector()) %>%
  dplyr::ungroup()

LPCs <- to_plot %>%
  dplyr::select(id, genotype, starts_with("LPC_") & !matches("\\.")) %>%
  dplyr::filter(!is.na(genotype))  %>%
  tidyr::pivot_longer(
    cols = starts_with("LPC_"),
    names_to = "species", values_to = "signal"
  ) %>%
  dplyr::group_by(species) %>%
  dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                log10 = log10(signal) %>% as.vector(),
                log10Z = scale(log10(signal)) %>% as.vector()) %>%
  dplyr::ungroup()

```

### Effect by species 

```{r}
### CRISPR data first  
CRISPR <- read.csv("pla_phyt_20.csv") %>%
  dplyr::mutate(geno = gsub("hom","mut", geno)) %>%
  # dplyr::mutate(geno = gsub("wt","AA", geno)) %>%
  dplyr::rename(genotype = geno)
sort(colnames(CRISPR))

CRISPR$PC_LPC <- CRISPR$PCs/CRISPR$LPCs

CRISPR <- CRISPR %>%
  dplyr::filter(genotype %in% c("wt","mut"))  %>%
  dplyr::mutate(genotype = factor(genotype, levels =  c("wt","mut")))


CRISPR_norm <- rbind(
  CRISPR %>%
    dplyr::select(sample_id, genotype, starts_with("PC.")) %>%
    tidyr::pivot_longer(cols = starts_with("PC."),
                        names_to = "species", values_to = "signal"
    ) %>%
    dplyr::group_by(species) %>%
    dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                  log10 = log10(signal) %>% as.vector(),
                  log10Z = scale(log10(signal)) %>% as.vector(),
                  pop = "CRISPR/CAS9",
                  trait = "PCs") %>%
    dplyr::ungroup(),
  
  CRISPR %>%
    dplyr::select(sample_id, genotype, starts_with("LPC.")) %>%
    tidyr::pivot_longer(
      cols = starts_with("LPC."),
      names_to = "species", values_to = "signal"
    ) %>%
    dplyr::group_by(species) %>%
    dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                  log10 = log10(signal) %>% as.vector(),
                  log10Z = scale(log10(signal)) %>% as.vector(),
                  pop = "CRISPR/CAS9",
                  trait = "LPCs")  %>%
    dplyr::ungroup()
)
```

# Now with the RILs


```{r}
cross <- read.cross(
  format = "csvs",  file = "B73xPT_AGPv3_genotype.csv",
  na.strings = c("-",NA),
  phefile = "B73xPT_combined_phenotype.csv",
  genotypes = c("A","B")) %>%
  convert2riself() %>%
  jittermap() %>%
  calc.genoprob( step = 1)

pheno <- read.csv("B73xPT_combined_phenotype.csv") %>%
  dplyr::group_by(id)


S3_8542287 <-factor(pull.geno(cross,3)[, "S3_8542287"])
levels(S3_8542287) <-  c("B73", "PT")

RILs <- dplyr::inner_join(
  data.frame( id = cross$pheno$id,
              genotype = S3_8542287),
  pheno
)

RILs$PC_LPC  <- log10(to_plot$PCs.LPCs)


RILs_norm <-rbind(
  RILs %>%
    dplyr::select(sample_id = id, genotype, starts_with("PC_") & !matches("LPC") & !matches("\\.")) %>%
    dplyr::filter(!is.na(genotype)) %>%
    tidyr::pivot_longer(cols = starts_with("PC_"),
                        names_to = "species", values_to = "signal"
    ) %>%
    dplyr::group_by(species) %>%
    dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                  log10 = log10(signal) %>% as.vector(),
                  log10Z = scale(log10(signal)) %>% as.vector(),
                  pop = "RILs",
                  trait = "PCs")  %>%
    dplyr::ungroup(),
  
  RILs %>%
    dplyr::select(sample_id = id, genotype, starts_with("LPC_") & !matches("\\.")) %>%
    dplyr::filter(!is.na(genotype))  %>%
    tidyr::pivot_longer(
      cols = starts_with("LPC_"),
      names_to = "species", values_to = "signal"
    ) %>%
    dplyr::group_by(species) %>%
    dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                  log10 = log10(signal) %>% as.vector(),
                  log10Z = scale(log10(signal)) %>% as.vector(),
                  pop = "RILs",
                  trait = "LPCs")  %>%
    dplyr::ungroup()
)


paired <-  rbind(CRISPR_norm, RILs_norm) %>%
  dplyr::group_by(pop,trait,genotype,species) %>%
  dplyr::summarise(signal = mean(signal, na.rm =TRUE),
                   signalZ = mean(signalZ, na.rm =TRUE)) %>%
  dplyr::ungroup() %>%
  dplyr::group_by(pop,genotype) %>%
  dplyr::filter(!is.na(signal)) %>%
  dplyr::ungroup() %>%
  dplyr::group_by(species) %>%
  dplyr::arrange(species,genotype) %>%
  dplyr::mutate( n = length(genotype)) %>%
  dplyr::filter( n == 2) %>%
  dplyr::mutate( sign = sign(signal[2] - signal[1])) %>% print(n=60)


paired$label <- ""
paired$empty_label <- ""
# paired$label[paired$genotype == "wt"] <- paired$species[paired$genotype == "mut"]

paired$label[paired$species == "PC..36.2." & paired$genotype == "wt"] <- "PC 36:2"
paired$label[paired$species == "LPC..18.2." & paired$genotype == "wt"] <- "LPC 18:2"

paired$label[paired$species == "PC_36_2" & paired$genotype == "B73"] <- "PC 36:2"
paired$label[paired$species == "LPC_18_2" & paired$genotype == "B73"] <- "LPC 18:2"
# paired$label[paired$genotype == "B73"] <- paired$species[paired$genotype == "PT"]

paired$empty_label[paired$genotype == "wt"] <- " "
paired$empty_label[paired$genotype == "B73"] <- " "

paired$trait <- factor(paired$trait, levels = c("PCs","LPCs"))
paired$pop <- factor(paired$pop, levels = c("RILs","CRISPR/CAS9"))


y_pos <- paired %>%
  dplyr::filter(pop == "RILs")  %>%
  dplyr::group_by(pop, trait) %>% 
  rstatix::get_y_position(signalZ ~ genotype) %>%
  dplyr::pull(y.position)
y_pos[2] <- y_pos[2] + 0.23  

wilcox_1 <- rbind(
  paired %>%
    dplyr::filter(pop == "RILs", trait == "PCs")  %>%
    dplyr::group_by(pop, trait) %>%
    rstatix::wilcox_test(signalZ ~ genotype, paired =TRUE, exact = TRUE, alternative = "less")%>%
    rstatix::add_xy_position(x = "genotype"),
  paired %>%
    dplyr::filter(pop == "RILs", trait == "LPCs")  %>%
    dplyr::group_by(pop, trait) %>%
    rstatix::wilcox_test(signalZ ~ genotype, paired =TRUE, exact = TRUE, alternative = "greater")%>%
    rstatix::add_xy_position(x = "genotype")
) %>% 
  rstatix::adjust_pvalue(method = "BH") %>%
  rstatix::add_significance()  %>%
  dplyr::mutate( p_raw = p, p = p.adj,
                 pval_fmt = format.pval(p.adj, digits = 2)) %>%
  dplyr::mutate(y.position = y_pos)
as.data.frame(wilcox_1)
levels(paired$pop)
?labeller
facet_labeller <- ggplot2::labeller(pop = c("RILs" ="RILs 3@8.51")) 

p1 <- paired %>%
  dplyr::filter(pop == "RILs") %>%
  ggplot(aes( x = genotype, y = signalZ, group = species, color = factor(sign))) +
  geom_point() + geom_line() +
  coord_cartesian(ylim = c(-1.5, 1.8))  +
  ylab("Signal (Z-score)") + xlab("Genotype")   +
  ggplot2::scale_color_manual(values= c("#df1b23","#1260ad") )+
  ggplot2::facet_grid(pop ~ trait, scales = "free", 
                      labeller = facet_labeller ) +
  ggpubr::stat_pvalue_manual(
    wilcox_1 , label = "p = {pval_fmt}", bracket.nudge.y = 0.1,
  ) +
  ggrepel::geom_text_repel(
    data= paired %>%
      dplyr::filter(pop == "RILs" & species == "PC_36_2" ),
    aes(label = label),
    hjust = 2,
    min.segment.length = unit(0, 'lines'),
    segment.size = 0.5,
    segment.alpha = 0.5,
    xlim  = c(1.2,NA),
    ylim  = c(NA,-1.2),
  ) +
  ggrepel::geom_text_repel(
    data= paired %>%
      dplyr::filter(pop == "RILs" & species == "LPC_18_2" ),
    aes(label = label),
    hjust = 2,
    min.segment.length = unit(0, 'lines'),
    segment.size = 0.5,
    segment.alpha = 0.5,
    xlim  = c(1.2,NA),
    ylim  = c(0.4,NA),
  ) +
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  theme(strip.text = element_text(size = base_size),
        strip.background =element_rect(color ="white",  fill="white"),
        axis.title.x=element_blank())


wilcox_2 <- rbind(
  paired %>%
    dplyr::filter(pop == "CRISPR/CAS9", trait == "PCs")  %>%
    dplyr::group_by(pop, trait) %>%
    rstatix::wilcox_test(signalZ ~ genotype, paired =TRUE, exact = TRUE, alternative = "less") %>%
    rstatix::add_xy_position(x = "genotype"),
  
  paired %>%
    dplyr::filter(pop == "CRISPR/CAS9", trait == "LPCs")  %>%
    dplyr::group_by(pop, trait) %>%
    rstatix::wilcox_test(signalZ ~ genotype, paired =TRUE, exact = TRUE, alternative = "greater") %>%
    rstatix::add_xy_position(x = "genotype")
  
) %>%    
  rstatix::adjust_pvalue(method = "BH") %>%
  rstatix::add_significance() %>%
  dplyr::mutate( p_raw = p, p = p.adj,
                 pval_fmt = format.pval(p.adj, digits = 2))


p2 <- paired %>%
  dplyr::filter(pop == "CRISPR/CAS9") %>%
  ggplot(aes( x = factor(genotype, levels = c("wt","mut")), 
              y = signalZ, group = species, color = factor(sign))) +
  geom_point() + geom_line() +
  coord_cartesian( ylim = c(-0.5, 0.55))  +
  ylab("Signal (Z-score)") + xlab("Genotype")   +
  ggplot2::scale_color_manual(values= c("#df1b23","#1260ad"))+
  ggplot2::facet_grid(pop ~ trait, scales = "free" ) +
  scale_x_discrete(labels = abbreviate) +
  ggpubr::stat_pvalue_manual(
    wilcox_2 , label = "p = {pval_fmt}", bracket.nudge.y = 0.05,
  ) +
  ggrepel::geom_text_repel(
    aes(label = label),
    direction = "y",
    hjust = 2,
    min.segment.length = unit(0, 'lines'),
    segment.size = 0.5,
    segment.alpha = 0.5,
    xlim  = c(1.2,NA),
  ) +
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  theme(strip.text = element_text(size = base_size),
        strip.background =element_rect(color ="white", fill="white"),
        axis.title.x=element_blank())

p3  <- ggpubr::ggarrange(p1,p2, align = "hv", nrow = 2)

# quartz(height = 1.3*4.13, width = 9.58)

png(file = "Figure3C.png", height = 538, width = 958))
ggpubr::ggarrange(
  ggpubr::ggarrange(p032,  p132, p320, 
                    p031,  p131, p310,
                    nrow = 2, ncol = 3),
  p3, 
  ncol = 2,
  widths = c(1.75,1), align = "hv")
dev.off()
```

