---
title: "fig6_teo_introgression"
author: "rra"
date: "5/4/2020"
output: html_document
---

```{r}
rm(list = ls())
library(tidyverse)
library(data.table)
library(cowplot)
library(wesanderson)
library(ggplot2)
library(MASS) 
library(qtl)
library(lme4) 
library(arm) 
library(lsmeans) 
library(pbkrtest)
library(cowplot) 
library(viridis)
library(qtlcharts)
library(readr)
library(ggfortify)
library(ggmap)
library(maps)
library(qtlcharts)
library(Hmisc)
library(pegas)
```


## Panel 6A alignment

```{r}
library(dplyr)
library(ape)
library(msa)
library(Biostrings)

fasta <- file.path(data_dir,"ZmPLA12_cds.fasta")

seqs <- readDNAStringSet(fasta)
aln<- as(msa(seqs, order = "input"), "BStringSet")

# search for alignment around lid domain variant
focus <- stringr::str_locate(as.character(aln[1,]), "CACCATCA")

# Specify codons around lid domain variant
focus[1,1] <- focus[1,1] - 2 - 3*3
focus[1,2] <- focus[1,2] + 2 + 3*4

mut <-  subseq(aln, start = focus[1,1], end = focus[1,2]) %>%
         as("DNAStringSet") %>%
         Biostrings::translate()

pop_levels <-  c("reference","parviglumis","mexicana",  
          "SW_US", "Mex_Highland","Gua_high","Mex_Lowland", 
          "SA_high", "SA_lowland")

seq_names <- data.frame(name= gsub("^\\s+|\\s+$","", names(mut))) %>%
  tidyr::separate(name, c("id","pop"), sep = "_|\\ ", 
                extra = "merge",
                remove = FALSE) %>%
  tibble::rowid_to_column("idx") 
             

names(mut) <-  seq_names$id

seq_names <- seq_names %>%
  dplyr::mutate(pop= factor(pop, levels = pop_levels )) %>%
  dplyr::arrange(pop,id)


## call unified interface msa() for default method (ClustalW) and
## default parameters

mut_aln <- msa(mut[seq_names$idx], order = "input")
## show resulting LaTeX code with default settings
msaPrettyPrint(mut_aln, output="asis", askForOverwrite=FALSE)

## create PDF file according to some custom settings
## its a PDF because TexShade is a latex package
## npo PNG output can be generated

out_file <-  "Fi6A.pdf"


msaPrettyPrint(mut_aln, file=out_file, output="pdf",
               showNames="left", showNumbering="none", showLogo="none",
               furtherCode=c("\\shadingmode{diverse}",
                             "\\nomatchresidues{Blue}{White}{upper}{up}",
                             "\\similarresidues{Blue}{White}{upper}{up}",
                             "\\shaderegion{1}{6..6}{Red}{White}"),
               showLegend=FALSE,
               showConsensus="none", verbose=FALSE, askForOverwrite=FALSE)

```

fd from palomeros and mushito

```{r}
fd_chr3 <- read_csv("data/fig_5/fd_chr_3_pts.csv")

names(fd_chr3)[7] <- "fd"
	
fd_chr3 %>% 
	#filter(`Yes-No`=="S") %>%
	ggplot(aes(x = start/10e5, xend = end/10e5, y = fd, yend = fd, colour = fd)) +
	scale_color_viridis() +
	geom_segment(size = 2) +
	ylim(0,1) +
  xlab("Chr 3 Mb") +
	geom_segment(x = 1000000, xend = 21000000, y = 0, yend = 0, colour = "black") +
  theme_cowplot(14)

fd_chr3 %>% 
	filter(POS > 750000) %>% 
	filter(POS < 20000000) %>%
	filter(`Yes-No`=="S") %>%
	ggplot(aes(x = start, xend = end, y = fd, yend = fd, colour = fd)) +
	scale_color_viridis() +
	geom_segment(size = 2) +
	ylim(0,1) +
	geom_vline(xintercept = 8542106, colour = "red", linetype="dotted", size = 1) +
  theme_cowplot(14)


```

# Haplotype Network

```{r}
# Getting georeferenced data
accn_file <- file.path('data','fig_2','germinate_SEED_GWAS_GBS_4022.tab')
accn_info <- read.table(file = accn_file, header = TRUE, quote = "",sep ="\t")
nrow(accn_info)

SEED_GWAS_GID <- read.table(file = file.path('data','fig_2', "SEED_GWAS_GID.tab"),
                            header = TRUE, quote = "",sep ="\t")

# There are one or more GWAS seed identifiers per general identifier in the table

GWAS_ID <- read.table(file = file.path('data','fig_2',"GWAS_ID.tab"), quote = "",sep ="\t")
GWAS_ID$taxa <- paste(GWAS_ID$V1,GWAS_ID$V2,GWAS_ID$V3,GWAS_ID$V4, sep = ":")

colnames(GWAS_ID)[1] <-"sample_id"

GWAS_ID <- GWAS_ID %>% 
  dplyr::select(sample_id,taxa) %>%
  inner_join(SEED_GWAS_GID)

mex <- accn_info %>% 
  dplyr::filter(!is.na(locations_longitude) & !is.na(locations_longitude)) %>%
  dplyr::filter(countries_country_code3 =="MEX") %>% # use the geolocator to retrieve this info
  dplyr::select(accn_GID  ="general_identifier",collnumb, lon ="locations_longitude",
                lat="locations_latitude",  ele="locations_elevation", country = "countries_country_code3") 

# Selecting just Mexican accessions (SEEDs) 
assoc <- mex %>% 
  dplyr::inner_join(GWAS_ID) %>% dplyr::select(taxa,collnumb) %>%
  dplyr::inner_join(traits) %>% 
  dplyr::arrange(locations_elevation) %>%
  dplyr::group_by(locations_elevation)

# Using a combination of tassel and shell cmds I:
# Merged SEEDs data with HapMap, which required:
# Download data from hapmap 282 set in AGPV2 for ZmPLA1.2
# Merge with SEDDs data for ZmPLA1.2 that is in AGPv2
# Liftover to AGPV3
# Filter heterozygote accessions out
# In order to use the Haplotype network plotting function
# I had to add phasing replacing '/' with '|' in the genotypes
# From 56 initial variants found in SEEDs down to 11 sites


vcf_file <-  file.path('data','fig_4', 'zmpla12_merged_het0_AGPv3_phased.vcf')

vcf <- vcfR::read.vcfR(vcf_file) 
ncol(vcf@gt) - 1

is_mexican <- colnames(vcf@gt) %in% c("FORMAT",assoc$taxa) 
is_teosinte <- grepl("TIL.*|Teo",colnames(vcf@gt), perl = TRUE)

vcf <-vcf[,is_mexican | is_teosinte ]
ncol(vcf@gt) - 1

# 916 mexican  out of  1379 initial
dna <- vcfR::vcfR2DNAbin(vcf)
h <- haplotype(dna, what="label")


haptab <- stack(setNames(attr(h, "index"), rownames(h))) %>%
  dplyr::mutate(h = as.numeric(ind)) %>%
  dplyr::select(h, label =ind, dnaid= values)

sumhap <- haptab %>% 
  dplyr::group_by(h,label) %>%
  dplyr::summarise(count= length(h))  %>%
  dplyr::arrange(-count) %>% ungroup() %>%
  dplyr::mutate(countrank = row_number()) %>%
  dplyr::mutate(cumcount = cumsum(count)) %>% 
  dplyr::mutate(cumfreq  = cumcount/sum(count)) %>% 
  dplyr::mutate(is_major = cumfreq < 0.9) %>%  # haplotypes covering 90% of the population fro the plot
  as.data.frame()

majorhid <- sumhap$h[sumhap$is_major]
majordnaid  <- haptab$dnaid[haptab$h %in% majorhid ]

majordna <- dna[majordnaid,]

h <- haplotype(majordna) # Here we get the sequences as haplotypes for plotting

check_mexicana <- function(x,h,dna) {
  h_names <- rownames(dna)[attr(h,"index")[[x]]]
  if(any(grepl("TIL08|TIL25",h_names))) {x} else {NULL}
}


hapind <- stack(setNames(attr(h, "index"), rownames(h))) %>% 
  dplyr::mutate(h = as.numeric(ind)) %>%
  dplyr::select(h, label =ind, dnaid = values) %>%
  dplyr::mutate(seqname = rownames(majordna)[dnaid]) %>%
  dplyr::mutate(seq = as.alignment(majordna)$seq) %>%
  dplyr::mutate(taxa = gsub("_.*","",seqname,perl = TRUE)) %>%
  dplyr::left_join(assoc) %>%
  dplyr::mutate(is_mexicana    = grepl("TIL08|TIL25", taxa, perl = TRUE)) %>%
  dplyr::mutate(is_parviglumis = grepl("TIL|Teo", taxa, perl = TRUE) & !is_mexicana) 


hapind$ele_class <- cut(
  hapind$locations_elevation, breaks = c(0,1000,2000,4000), 
  labels = c("low","mid","high"))

hapind$pop <- hapind$ele_class 
hapind$pop <- factor(hapind$pop, levels = c("low","mid","high"))

pie_tab <- with(hapind, table(hap=label, pop=pop))

net <- pegas::haploNet(h)

pal <- c("#CC6677","#BCDCAD", "#88CCEE")


# Preliminary plot just for getting layout coordinates
p <- plot(net, legend = FALSE,
     size=attr(net, "freq")/50, 
     scale.ratio = 2, cex = 0., 
     pie= pie_tab,
     col = "white",
     bg=pal,
     show.mutation = 2,threshold =0 )

# Parameters for adjusting pie labels (manually):
size.adjust <- c(0.8, 0.6,0.7, 1.5,1.5,0.6,1,1.2,1.5)
angle.adjust <- -c(pi/3,pi/3,pi/3,-pi/2, pi/2, pi/3, -pi/3, -pi/2,pi)

# Final plot
pdf(file.path('output',"fig_1B.pdf"))
plot(net, legend = FALSE,
     labels = FALSE,
     size=attr(net, "freq")/50, 
     scale.ratio = 2, cex = 0., 
     pie= pie_tab,
     bg=pal,
     show.mutation = 2,threshold =0, )
# Adjusting pie labels
text(p$xx + cos(angle.adjust)*p$size*size.adjust , 
     p$yy + sin(angle.adjust)*p$size*size.adjust ,
     labels = p$labels, srt = 90)
legend(-25,30, box.lty = 0, colnames(pie_tab), fill=pal,
       title = "Population by Elevation", ncol =3)
dev.off()

```

## PLA1.2 282 expression vs Flowering time

```{r}
library(dplyr)
library(tibble)
library(ggplot2)
library(tidyselect)
library(ggpubr)
library(ape)
library(phytools)

data_dir <- "../data/fig_5"

# Make phenotype table #########################################################
# Summer - winter data Hung et al 2012

# https://datacommons.cyverse.org/browse/iplant/home/shared/panzea/phenotypes/Hung_etal_2012_PNAS_data-120523.zip
gddtf <- read.csv("Dataset_S2.Photoperiod_paper_data_supplement_line_BLUPs.csv") %>%
  dplyr::filter(pop==27) %>%
  dplyr::select( Taxa = geno_code, gdd_anth_long, gdd_anth_short,gdd_anth_photo_resp
                 , gdd_silk_long, gdd_silk_short, gdd_silk_photo_resp)

# https://datacommons.cyverse.org/browse/iplant/home/shared/commons_repo/curated/Kremling_Nature3RNASeq282_March2018

tissue_file <- list.files(pattern = "TASSEL.*Dups.txt") # This dataset is huge add to .gitignore
tissue <-  gsub(".*and_Altnames_","",tissue_file)
tissue <-  gsub("_.*","",tissue)
names(tissue_file) <- tissue
tissue  <- c(tissue[c(1,2,4,5,3)], NA,tissue[c(6,7)])


RNA <-read.table(file=tissue_file["GShoot"], skip =2, header = TRUE)
RNA$Taxa <- gsub("282set_","",RNA$Taxa)

DTF_RNA <- gddtf %>%
  dplyr::inner_join(RNA) %>%
  dplyr::select(Taxa, gdd_anth_long, gdd_anth_short, gdd_anth_photo_resp, gdd_silk_long, gdd_silk_short, gdd_silk_photo_resp,everything())

colnames(DTF_RNA) <- gsub("gdd_","",colnames(DTF_RNA))

# hapmap data from Panzea AGPv3 282 set genotypes for GRMZM2G353444 cds

hmp <- file.path(data_dir,'GRMZM2G353444_AGPV3_282.hapmap.hmp.txt')

tree <- read.tree("ZmPLA1.2_282_til_german_AA.tree")
class(tree)

tree$tip.label
quartz()
plotTree(tree)
nodelabels()

PT_lineage <- extract.clade(tree, 317)
B73_lineage <- extract.clade(tree, 336)
PT_lineage$tip.label

quartz()
plotTree(B73_lineage)

quartz()
plotTree(PT_lineage)

gt <- read.table(hmp, skip =1)

con <- file(hmp)
first_line <- readLines(con,n=1)
close(con)

h <- strsplit(first_line,"\t") %>% unlist

h <- gsub("#|282set_","", h) 
h %>%  length()
colnames(gt)  <- h 

pla12_lpcat <- DTF_RNA %>%
  dplyr::select(Taxa, anth_long, anth_short,
                anth_photo_resp, silk_long, 
                silk_short, silk_photo_resp,GRMZM2G353444,GRMZM2G481755)
gtm <- gt %>% 
  dplyr::filter(grepl("/",alleles) ) %>%
  dplyr::select(-(alleles:QCcode)) %>% t()
colnames( gtm) <- paste0("S",gtm[1,])
gtm <- gtm[-1,]
gtm <-  cbind(rownames(gtm),gtm)
colnames(gtm)[1] <- "Taxa"
as.data.frame(gtm)

# adding lineage inforrmation
clade_RNA_FT <- as.data.frame(gtm) %>% left_join(pla12_lpcat)

clade_RNA_FT$lineage <- NULL
clade_RNA_FT$lineage[clade_RNA_FT$Taxa %in% PT_lineage$tip.label] <- "PT"
clade_RNA_FT$lineage[clade_RNA_FT$Taxa %in% B73_lineage$tip.label] <- "B73"

#------------------------------------------------------------------

toplot <- clade_RNA_FT %>%
  dplyr::filter(!is.na(lineage)) 

p1 <- toplot  %>%
  ggplot2::ggplot( aes( x = GRMZM2G353444, 
                        y = anth_long)
  ) +
  xlab("ZmPLA1.2") +
  ylab("Long Days DTA (gdd)") +
  labs(color = "Lineage") + 
  ggplot2::geom_point(aes( group = lineage,
                           color = lineage))  +
  ggplot2::geom_smooth(method = "lm", se=FALSE, col = "black") +
  ggpubr::stat_cor(label.sep = "\n",label.x.npc = 0.75,label.y.npc = 0.1)  +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad")) +
  ggpubr::theme_pubr(legend = "none") 


# Marginal densities along x axis

xdens <- ggboxplot(toplot, "lineage", "GRMZM2G353444",
                   color = "lineage", palette = c("#df1b23","#1260ad"), alpha = 0.5) +
  ggplot2::coord_flip()

# Marginal densities along y axis
# Need to set coord_flip = TRUE, if you plan to use coord_flip()

ydens  <- ggboxplot(toplot, "lineage", "anth_long",
                    color = "lineage", palette = c("#df1b23","#1260ad"), alpha = 0.5) 

p12 <- cowplot::insert_xaxis_grob(p1, xdens, grid::unit(.2, "null"), position = "top")
p13 <- cowplot::insert_yaxis_grob(p12, ydens, grid::unit(.2, "null"), position = "right")

p2 <- toplot  %>%
  ggplot2::ggplot( aes( x = GRMZM2G353444, 
                        y = anth_short)
  ) +
  xlab("ZmPLA1.2") +
  ylab("Short Days DTA (gdd)") +
  ggplot2::geom_point(aes( group = lineage,
                           color = lineage))  +
  ggplot2::geom_smooth(method = "lm", se=FALSE, col = "black") +
  ggpubr::stat_cor(label.sep = "\n",label.x.npc = 0.75,label.y.npc = 0.1)  +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad")) +
  ggpubr::theme_pubr(legend="none") 

# Marginal densities along x axis
xdens <- ggboxplot(toplot, "lineage", "GRMZM2G353444",
                   color = "lineage", palette = c("#df1b23","#1260ad"), alpha = 0.5) +
  ggplot2::coord_flip()

# Marginal densities along y axis
# Need to set coord_flip = TRUE, if you plan to use coord_flip()
ydens  <- ggboxplot(toplot, "lineage", "anth_short",
                    color = "lineage", palette = c("#df1b23","#1260ad"), alpha = 0.5)

p22 <- cowplot::insert_xaxis_grob(p2, xdens, grid::unit(.2, "null"), position = "top")

p23 <- cowplot::insert_yaxis_grob(p22, ydens, grid::unit(.2, "null"), position = "right")


quartz(height= 1.5*4.72, width =1.5*2.3)
ggpubr::ggarrange(cowplot::ggdraw(p13),cowplot::ggdraw(p23),
                  ncol = 1,
                  common.legend = TRUE,
                  align = "hv")

#------------------------------------------------------------------
toplot <- clade_RNA_FT %>%
  dplyr::filter(!is.na(S3_8543470)) %>%
  dplyr::filter(!S3_8543470 == "N") 

p1 <- toplot  %>%
  ggplot2::ggplot( aes( x = GRMZM2G353444, 
                        y = anth_long)
  ) +
  xlab("ZmPLA1.2") +
  ylab("Long Days DTA (gdd)") +
  ggplot2::geom_point(aes( group = S3_8543470,
                           color = S3_8543470))  +
  ggplot2::geom_smooth(method = "lm", se=FALSE, col = "black") +
  ggpubr::stat_cor()  +
 scale_color_manual(values = c("#1260ad", "#df1b23","#68ac57")) +
  ggpubr::theme_pubr(legend = "none") 


# Marginal densities along x axis

xdens <- ggboxplot(toplot, "S3_8543470", "GRMZM2G353444",
                   color = "S3_8543470",  palette = c("#df1b23","#1260ad","#68ac57"), alpha = 0.5) +
  ggplot2::coord_flip()

# Marginal densities along y axis
# Need to set coord_flip = TRUE, if you plan to use coord_flip()

ydens  <- ggboxplot(toplot, "S3_8543470", "anth_long",
                    color = "S3_8543470",  palette = c("#df1b23","#1260ad", "#68ac57"), alpha = 0.5) 

p12 <- cowplot::insert_xaxis_grob(p1, xdens, grid::unit(.2, "null"), position = "top")
p13 <- cowplot::insert_yaxis_grob(p12, ydens, grid::unit(.2, "null"), position = "right")

p2 <- toplot %>%
  ggplot2::ggplot( aes( x = GRMZM2G353444, 
                        y = anth_short)
  ) +
  ggplot2::xlab("ZmPLA1.2") +
  ggplot2::ylab("Short Days DTA (gdd)") +
  ggplot2::labs(color = "S3_8543470") + 
  ggplot2::geom_point(aes( group = S3_8543470,
                           color = S3_8543470))  +
  ggplot2::geom_smooth(method = "lm", se=FALSE, col = "black") +
  ggpubr::stat_cor() +
  scale_color_manual(values = c("#1260ad", "#df1b23","#68ac57")) +
  ggpubr::theme_pubr(legend = "none") 

# Marginal densities along x axis
xdens <- ggboxplot(toplot, "S3_8543470", "GRMZM2G353444",
                   color = "S3_8543470",  palette = c("#df1b23","#1260ad", "#68ac57"), alpha = 0.5) +
  ggplot2::coord_flip()

# Marginal densities along y axis
# Need to set coord_flip = TRUE, if you plan to use coord_flip()
ydens  <- ggboxplot(toplot, "S3_8543470", "anth_short",
                    color = "S3_8543470",  palette = c("#df1b23","#1260ad", "#68ac57"), alpha = 0.5)

p22 <- cowplot::insert_xaxis_grob(p2, xdens, grid::unit(.2, "null"), position = "top")

p23 <- cowplot::insert_yaxis_grob(p22, ydens, grid::unit(.2, "null"), position = "right")


ggpubr::ggarrange(cowplot::ggdraw(p13),cowplot::ggdraw(p23),
                  legend ="none",
                  nrow = 2,
                  common.legend = TRUE,
                  widths = c(1, 1.4))

```




