---
title: "Fig 3 Code"
output: html_document
editor_options:
  chunk_output_type: inline
---

```{r}
rm(list = ls())
library(tidyverse)
library(data.table)
library(cowplot)
library(wesanderson)
library(ggplot2)
library(MASS) 
library(qtl)
library(lme4) 
library(arm) 
library(lsmeans) 
library(pbkrtest)
library(cowplot) 
library(viridis)
library(qtlcharts)
library(readr)
library(ggfortify)
library(ggmap)
library(maps)
library(qtlcharts)
library(Hmisc)
library(pegas)
library(mdthemes)
```



# Annotated LOD profile

```{r}
library(dplyr)
library(qtl)
library(GenomicRanges)
library(ggbio)
library(igraph)
library(qtl)
source("detect_peaks.R")

##### QTL analysis ##########################################################
n_threads <- 5
n_perm <- 1000
seed <- 1234567890
set.seed(seed)
data_dir <- "../data/fig_3"
geno_file  <- file.path(data_dir, "B73xPT_AGPv3_genotype.csv")
pheno_file <- file.path(data_dir, "B73xPT_combined_phenotype.csv")

cross <- read.cross(
  format = "csvs",  file = geno_file,
  na.strings = c("-",NA),
  phefile = pheno_file,
  genotypes = c("A","B")) %>%
  convert2riself() %>%
  jittermap() %>%
  calc.genoprob( step = 1)

# Select phenotypes with up to 50% missing data ###############################
# missing phenotypes will generate errors

nind <- nind(cross)
nphe <- nphe(cross)

missing <- colSums(is.na(cross$pheno))
missing_thresh <- 0.5

pheno_col <- which( missing < missing_thresh * nind + 4) # this form comes from the readcross warning
                                                         # RILs with genotype but no phenotype
pheno_col <- pheno_col[-1]


single_scan <- scanone(cross, pheno.col = pheno_col, method = "hk")
perms <- scanone(cross, pheno.col = pheno_col,
                 n.perm = n_perm, method = "hk", n.cluster = n_threads)


peaks <- get_peak_table(perms = perms, single_scan = single_scan)

refined_peaks <- refine_peaks(peaks, perms = perms, single_scan = single_scan) %>%
  select(trait,lod,thresh_adj,thresh, everything()) %>%
  arrange(-lod) %>% tibble() %>%
  print(n = 150)

new_peaks  <- refined_peaks[refined_peaks$lod > refined_peaks$thresh_adj,]

# Variance explained by  3@8.5

pla12_pos <- pull.map(cross = cross)[[3]]["S3_8542287"]

q1 <- makeqtl(sim.geno(cross), 3, pla12_pos, "3@8.5")

qtraits <-  c("PCs","LPCs","PCs.LPCs")

peaks %>%
  dplyr::filter(trait %in% qtraits) %>%
  dplyr::select(trait, pos, marker, marker_lod)

trait_var <- lapply(qtraits, function(x){
  fit <- fitqtl(sim.geno(cross) , 
       pheno.col=which(colnames(cross$pheno) == x), 
       dropone = TRUE,
       q1)
  fit$result.full[1,5]
})

names(trait_var) <- qtraits
trait_var

##### Output Profile ##########################################################

p1 <- single_scan  %>%
  dplyr::select( chr,pos, LPCs, PCs,PCs.LPCs) %>%
  tidyr::pivot_longer(cols = c("LPCs", "PCs","PCs.LPCs"),
                      names_to = "trait",values_to = "LOD") %>%
  dplyr::mutate(trait = factor(trait, levels = c( "PCs","LPCs", "PCs.LPCs"))) %>%
  ggplot2::ggplot() +
  ggplot2::geom_line(aes(x = pos, y = LOD, col = trait))+
  ggplot2::xlab("Chromosome") +
  ggplot2::ylab("LOD") +
  ggplot2::geom_hline(aes(yintercept = LPCs_thresh), col = pal[1], lty =2)+
  ggplot2::facet_wrap(.~ as.factor(chr) ,
                      scales = "free_x",
                      strip.position = "bottom",
                      ncol = 10)  +
  ggplot2::geom_rug(aes(x = pos),sides="b", size =0.01) +
  ggpubr::theme_classic2(base_size = 18) +
  ggplot2::theme(legend.position = "none",
                 strip.background = ggplot2::element_blank(),
                  strip.placement = "outside",
                  axis.text.x = ggplot2::element_text(size =8, angle = 90))

p2 <- single_scan  %>%
  dplyr::filter(chr==3) %>%
  dplyr::select( chr,pos, LPCs, PCs,PCs.LPCs) %>%
  tidyr::pivot_longer(cols = c("LPCs", "PCs","PCs.LPCs"),
                      names_to = "trait",values_to = "LOD") %>%
  dplyr::mutate(trait = factor(trait, levels = c( "PCs","LPCs", "PCs.LPCs"))) %>%
  ggplot2::ggplot() +
  ggplot2::geom_line(aes(x = pos, y = LOD, col = trait))+
  ggplot2::xlab("Position cM") +
  ggplot2::ylab("") +
  ggplot2::geom_hline(aes(yintercept = LPCs_thresh), col = pal[1], lty =2)+
  ggplot2::geom_vline(aes(xintercept = pla12_pos), col = "red") +
  ggplot2::geom_rug(aes(x = pos),sides="b", size =0.1) +
  ggpubr::theme_classic2() +
  ggplot2::theme( plot.margin = unit(c(0,0,0,0), "lines"),
                  legend.position = "none")

plot.with.inset <-
  cowplot::ggdraw() +
  cowplot::draw_plot(p1) +
  cowplot::draw_plot(p2, x = 0.3, y = .45, width = .68, height = .45)


png(file = "Figure3A.png", height = 616, width = 1168)

# quartz(height = 2*3.08, width = 2*5.84)

print(plot.with.inset)
 
grid::grid.text(expression(italic("ZmPla1.2")),
                just = "left", x = 0.475, y = 0.92)
grid::grid.text(expression(italic("qPCs/LPCs")),
                just = "left", x = 0.525, y = 0.85)
grid::grid.text(expression(italic("qLPCs")),
                just = "left",  x = 0.54, y = 0.675)
grid::grid.text(expression(italic("qPCs")),
                just = "left", x = 0.55, y = 0.63)

grid::grid.lines(x = unit(c(0.512, 0.523), "npc"),
                 y = unit(c(0.85, 0.85), "npc"),
                 gp = grid::gpar(col= pal[3]))

grid::grid.lines(x = unit(c(0.51, 0.538), "npc"),
                 y = unit(c(0.656, 0.675), "npc"),
                 gp = grid::gpar(col= pal[2]))

grid::grid.lines(x = unit(c(0.515, 0.548), "npc"),
                 y = unit(c(0.60, 0.63), "npc"),
                 gp = grid::gpar(col= pal[1]))

grid::grid.lines(x = unit(c(0.255, 0.36), "npc"),
                 y = unit(c(0.215, 0.49), "npc"),
                 gp = grid::gpar(fill="black"),
                 arrow = grid::arrow(length = unit(0.1, "inches"),
                                     ends="last", type="closed"))
grid::grid.lines(x = unit(c(0.33, 0.95), "npc"),
                 y = unit(c(0.215, 0.49), "npc"),
                 gp = grid::gpar(fill="black"),
                 arrow = grid::arrow(length = unit(0.1, "inches"),
                                     ends="last", type="closed"))
dev.off()
```

## Effect Sizes.

```{r}
base_size <- 13

metabolite <- read.csv("pla_phyt_20.csv") %>%
  dplyr::mutate(geno = gsub("hom","mut", geno)) %>%
  dplyr::mutate(geno = gsub("wt","wt", geno)) %>%
  dplyr::rename(genotype = geno)


metabolite$PC_LPC <- metabolite$PCs/metabolite$LPCs
```

## Normalized per species

```{r}
plot_traits <- c("PCs","LPCs", "PC_LPC")

species_effect <- metabolite %>%
  dplyr::select(sample_id,genotype, all_of(plot_traits)) %>%
  dplyr::mutate_if(is.numeric, scale) %>%
  tidyr::pivot_longer(all_of(plot_traits), names_to = "species", values_to = "Zscore") %>%
  dplyr::filter(!is.na(genotype)) %>%
  dplyr::mutate(species = factor(species, levels = plot_traits)) 

species_effect$Zscore <- species_effect$Zscore[,1]

sp_label <- c("PCs","LPCs", "PC/LPC")
names(sp_label) <- plot_traits
facet_labeller <- ggplot2::labeller(species = sp_label) 

pe12 <- species_effect %>%
  dplyr::filter(!is.na(genotype)) %>%
  ggplot2::ggplot(aes(x = genotype, y = Zscore, color = genotype)) +
  ggplot2::geom_jitter(position=position_jitter(0.2), size = 1.5) +
  ggplot2::stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), 
                        geom="pointrange", color = "black", size = 0.5) +
  ggplot2::xlab("Genotype") + ggplot2::ylab("Z-score") +
  ggplot2::facet_wrap(. ~ species, scales = "free", 
                      labeller =facet_labeller,
                      ncol = 3)  +
  scale_x_discrete(labels = c(expression(italic("wt")), expression(italic("CR")))) +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad")) +
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  ggplot2::theme( axis.title = element_blank(),
                 strip.text = element_blank(),
                 strip.background =element_rect(color ="white",  fill="white"))
```

# Now with the RILs 
```{r}
cross <- read.cross(
  format = "csvs",  file = "B73xPT_AGPv3_genotype.csv",
  na.strings = c("-",NA),
  phefile = "B73xPT_combined_phenotype.csv",
  genotypes = c("A","B")) %>%
  convert2riself() %>%
  jittermap() %>%
  calc.genoprob( step = 1)

pheno <- read.csv("B73xPT_combined_phenotype.csv") %>%
    dplyr::group_by(id)

cross$pheno$id

S3_8542287 <-factor(pull.geno(cross,3)[, "S3_8542287"])
levels(S3_8542287) <-  c("B73", "PT")

to_plot <- dplyr::inner_join(
  data.frame( id = cross$pheno$id,
              genotype = S3_8542287),
  pheno
)

plot_traits <- c("PCs","LPCs", "PCs.LPCs")
species_effect <-  to_plot %>% 
  dplyr::select(id, genotype, all_of(plot_traits)) %>%
  dplyr::mutate_if(is.numeric, scale) %>%
  tidyr::pivot_longer(all_of(plot_traits), names_to = "species", values_to = "Zscore" )
species_effect$Zscore <- species_effect$Zscore[,1]

sp_label <- c("PCs","LPCs", "PCs/LPCs")
names(sp_label) <- plot_traits
facet_labeller <- ggplot2::labeller(species = sp_label) 

pe11 <- species_effect %>%
  dplyr::filter(!is.na(genotype)) %>%
  dplyr::mutate(species = factor(species, levels = plot_traits)) %>%
  ggplot2::ggplot(aes(x = genotype, y = Zscore, color = genotype)) +
  ggplot2::geom_jitter(position=position_jitter(0.2), size = 1.5) +
  ggplot2::stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), 
                        geom="pointrange", color = "black", size = 0.5) +
  ggplot2::facet_wrap(. ~ species, scales = "free", 
                      labeller = facet_labeller,
                      ncol = 3)  +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad")) +
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  ggplot2::theme( axis.title = element_blank(),
                 strip.text = element_blank())

quartz(height =2.5, width =6)

```

### Effect by species 

```{r}
### CRISPR data first  
CRISPR <- read.csv("pla_phyt_20.csv") %>%
  dplyr::mutate(geno = gsub("hom","mut", geno)) %>%
  # dplyr::mutate(geno = gsub("wt","AA", geno)) %>%
  dplyr::rename(genotype = geno)
sort(colnames(CRISPR))

CRISPR$PC_LPC <- CRISPR$PCs/CRISPR$LPCs

CRISPR <- CRISPR %>%
  dplyr::filter(genotype %in% c("wt","mut"))  %>%
  dplyr::mutate(genotype = factor(genotype, levels =  c("wt","mut")))


CRISPR_norm <- rbind(
  CRISPR %>%
    dplyr::select(sample_id, genotype, starts_with("PC.")) %>%
    tidyr::pivot_longer(cols = starts_with("PC."),
                        names_to = "species", values_to = "signal"
    ) %>%
    dplyr::group_by(species) %>%
    dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                  log10 = log10(signal) %>% as.vector(),
                  log10Z = scale(log10(signal)) %>% as.vector(),
                  pop = "CRISPR/CAS9",
                  trait = "PCs") %>%
    dplyr::ungroup(),
  
  CRISPR %>%
    dplyr::select(sample_id, genotype, starts_with("LPC.")) %>%
    tidyr::pivot_longer(
      cols = starts_with("LPC."),
      names_to = "species", values_to = "signal"
    ) %>%
    dplyr::group_by(species) %>%
    dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                  log10 = log10(signal) %>% as.vector(),
                  log10Z = scale(log10(signal)) %>% as.vector(),
                  pop = "CRISPR/CAS9",
                  trait = "LPCs")  %>%
    dplyr::ungroup()
)
```


```{r}
# Now with the RILs -----------------------------------------------------------------

library(qtl)

cross <- read.cross(
  format = "csvs",  file = "B73xPT_AGPv3_genotype.csv",
  na.strings = c("-",NA),
  phefile = "B73xPT_combined_phenotype.csv",
  genotypes = c("A","B")) %>%
  convert2riself() %>%
  jittermap() %>%
  calc.genoprob( step = 1)

pheno <- read.csv("B73xPT_combined_phenotype.csv") %>%
  dplyr::group_by(id)

cross$pheno$id

S3_8542287 <-factor(pull.geno(cross,3)[, "S3_8542287"])
levels(S3_8542287) <-  c("B73", "PT")

RILs <- dplyr::inner_join(
  data.frame( id = cross$pheno$id,
              genotype = S3_8542287),
  pheno
)

RILs$PC_LPC  <- log10(to_plot$PCs.LPCs)


RILs_norm <-rbind(
  RILs %>%
    dplyr::select(sample_id = id, genotype, starts_with("PC_") & !matches("LPC") & !matches("\\.")) %>%
    dplyr::filter(!is.na(genotype)) %>%
    tidyr::pivot_longer(cols = starts_with("PC_"),
                        names_to = "species", values_to = "signal"
    ) %>%
    dplyr::group_by(species) %>%
    dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                  log10 = log10(signal) %>% as.vector(),
                  log10Z = scale(log10(signal)) %>% as.vector(),
                  pop = "RILs",
                  trait = "PCs")  %>%
    dplyr::ungroup(),
  
  RILs %>%
    dplyr::select(sample_id = id, genotype, starts_with("LPC_") & !matches("\\.")) %>%
    dplyr::filter(!is.na(genotype))  %>%
    tidyr::pivot_longer(
      cols = starts_with("LPC_"),
      names_to = "species", values_to = "signal"
    ) %>%
    dplyr::group_by(species) %>%
    dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                  log10 = log10(signal) %>% as.vector(),
                  log10Z = scale(log10(signal)) %>% as.vector(),
                  pop = "RILs",
                  trait = "LPCs")  %>%
    dplyr::ungroup()
)


paired <-  rbind(CRISPR_norm, RILs_norm) %>%
  dplyr::group_by(pop,trait,genotype,species) %>%
  dplyr::summarise(signal = mean(signal, na.rm =TRUE),
                   signalZ = mean(signalZ, na.rm =TRUE)) %>%
  dplyr::ungroup() %>%
  dplyr::group_by(pop,genotype) %>%
  dplyr::filter(!is.na(signal)) %>%
  dplyr::ungroup() %>%
  dplyr::group_by(species) %>%
  dplyr::arrange(species,genotype) %>%
  dplyr::mutate( n = length(genotype)) %>%
  dplyr::filter( n == 2) %>%
  dplyr::mutate( sign = sign(signal[2] - signal[1])) %>% print(n=60)

table(paired$genotype,paired$species)

paired$label <- ""
paired$empty_label <- ""
# paired$label[paired$genotype == "wt"] <- paired$species[paired$genotype == "mut"]

paired$label[paired$species == "PC..36.2." & paired$genotype == "mut"] <- "PC36:2"
paired$label[paired$species == "PC..36.5." & paired$genotype == "mut"] <- "PC36:5"
paired$label[paired$species == "LPC..18.2." & paired$genotype == "mut"] <- "LPC18:2"
paired$label[paired$species == "LPC..18.1." & paired$genotype == "mut"] <- "LPC18:1"


paired$label[paired$species == "PC_36_2" & paired$genotype == "PT"] <- "PC36:2"
paired$label[grepl("PC_36_5",paired$species) & paired$genotype == "PT"] <-  c("PC36:5A","PC36:5B")
paired[grepl("PC_36_5",paired$species) & paired$genotype == "PT",]
paired$label[paired$species == "LPC_18_1" & paired$genotype == "PT"] <- "LPC18:1"

paired$label[paired$species == "LPC_18_2" & paired$genotype == "PT"] <- "LPC18:2"

paired$species
paired$empty_label[paired$genotype == "wt"] <- " "
paired$empty_label[paired$genotype == "B73"] <- " "

paired$trait <- factor(paired$trait, levels = c("PCs","LPCs"))
paired$pop <- factor(paired$pop, levels = c("RILs","CRISPR/CAS9"))


y_pos <- paired %>%
  dplyr::filter(pop == "RILs")  %>%
  dplyr::group_by(pop, trait) %>% 
  rstatix::get_y_position(signalZ ~ genotype) %>%
  dplyr::pull(y.position)
y_pos[2] <- y_pos[2] + 0.23  

wilcox_1 <- rbind(
  paired %>%
    dplyr::filter(pop == "RILs", trait == "PCs")  %>%
    dplyr::group_by(pop, trait) %>%
    rstatix::wilcox_test(signalZ ~ genotype, paired =TRUE, exact = TRUE, alternative = "less")%>%
    rstatix::add_xy_position(x = "genotype"),
  paired %>%
    dplyr::filter(pop == "RILs", trait == "LPCs")  %>%
    dplyr::group_by(pop, trait) %>%
    rstatix::wilcox_test(signalZ ~ genotype, paired =TRUE, exact = TRUE, alternative = "greater")%>%
    rstatix::add_xy_position(x = "genotype")
) %>% 
  rstatix::adjust_pvalue(method = "BH") %>%
  rstatix::add_significance()  %>%
  dplyr::mutate( p_raw = p, p = p.adj,
                 pval_fmt = format.pval(p.adj, digits = 2)) %>%
  dplyr::mutate(y.position = y_pos)
as.data.frame(wilcox_1)
levels(paired$pop)



paired  %>%
  dplyr::filter(pop == "RILs") %>% print(n= 100)

pe21 <- paired %>%
  dplyr::filter(pop == "RILs" & trait == "PCs") %>%
  ggplot(aes( x = genotype, y = signalZ, group = species, color = factor(sign))) +
  geom_point() + geom_line() +
  coord_cartesian(ylim = c(-1.5, 1.8), xlim = c(1,2.5))  +
  ylab("Signal (Z-score)") + xlab("Genotype")   +
  ggplot2::scale_color_manual(values= c("#df1b23","#1260ad") )+
  # ggplot2::facet_wrap(pop ~ trait, scales = "free_y") +
  ggpubr::stat_pvalue_manual(
    wilcox_1 %>% dplyr::filter(trait == "PCs"), label = "p = {pval_fmt}", bracket.nudge.y = 0.1,
  ) +
  ggrepel::geom_text_repel(
    data= paired  %>%
      dplyr::filter(pop == "RILs" & species %in% c("PC_36_2","PC_36_5_A", "PC_36_5_B")),
    aes(label = label),
    hjust = 2,
    min.segment.length = unit(0, 'lines'),
    segment.size = 0.5,
    segment.alpha = 0.5,
    xlim  = c(2.2, NA),
  ) +
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  theme(strip.text = element_blank(),
        axis.title=element_blank(),
  )

pe22 <- paired %>%
  dplyr::filter(pop == "RILs" & trait == "LPCs") %>%
  ggplot(aes( x = genotype, y = signalZ, group = species, color = factor(sign))) +
  geom_point() + geom_line() +
  coord_cartesian(ylim = c(-1.5, 1.8), xlim = c(1,2.5))  +
  ylab("Signal (Z-score)") + xlab("Genotype")   +
  ggplot2::scale_color_manual(values= c("#df1b23","#1260ad") )+
  # ggplot2::facet_wrap(pop ~ trait, scales = "free_y") +
  ggpubr::stat_pvalue_manual(
    wilcox_1 %>% dplyr::filter(trait == "LPCs"), label = "p = {pval_fmt}", bracket.nudge.y = 0.1,
  ) +
  ggrepel::geom_text_repel(
    data= paired  %>%
       dplyr::filter(pop == "RILs" & species %in% c("LPC_18_1",  "LPC_18_2")),
    aes(label = label),
    hjust = 2,
    min.segment.length = unit(0, 'lines'),
    segment.size = 0.5,
    segment.alpha = 0.5,
    xlim  = c(2.2, NA),
  ) +
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  theme(strip.text = element_blank(),
        axis.title=element_blank(),
  )


wilcox_2 <- rbind(
  paired %>%
    dplyr::filter(pop == "CRISPR/CAS9", trait == "PCs")  %>%
    dplyr::group_by(pop, trait) %>%
    rstatix::wilcox_test(signalZ ~ genotype, paired =TRUE, exact = TRUE, alternative = "less") %>%
    rstatix::add_xy_position(x = "genotype"),
  
  paired %>%
    dplyr::filter(pop == "CRISPR/CAS9", trait == "LPCs")  %>%
    dplyr::group_by(pop, trait) %>%
    rstatix::wilcox_test(signalZ ~ genotype, paired =TRUE, exact = TRUE, alternative = "greater") %>%
    rstatix::add_xy_position(x = "genotype")
  
) %>%    
  rstatix::adjust_pvalue(method = "BH") %>%
  rstatix::add_significance() %>%
  dplyr::mutate( p_raw = p, p = p.adj,
                 pval_fmt = format.pval(p.adj, digits = 2))

pe23 <- paired %>%
  dplyr::filter(pop == "CRISPR/CAS9" & trait == "PCs") %>%
  ggplot2::ggplot(aes( x = genotype, y = signalZ, group = species, color = factor(sign))) +
  ggplot2::geom_point() + geom_line() +
  ggplot2::coord_cartesian( ylim = c(-0.5, 0.6), xlim = c(1,2.5))  +
  ggplot2::ylab("Signal (Z-score)") + ggplot2::xlab("Genotype")   +
  ggplot2::scale_color_manual(values= c("#df1b23","#1260ad") ) +
  ggplot2::scale_x_discrete(labels = c(expression(italic("wt")), expression(italic("CR")))) +
  # ggplot2::facet_wrap(pop ~ trait, scales = "free_y") +
  ggpubr::stat_pvalue_manual(
    wilcox_2 %>% dplyr::filter(trait == "PCs"), label = "p = {pval_fmt}", bracket.nudge.y = 0.1,
  ) +
  ggrepel::geom_text_repel(
    data= paired  %>%
      dplyr::filter(pop == "CRISPR/CAS9" & species %in% c("PC..36.2.","PC..36.5.")),
    aes(label = label),
    hjust = 2,
    min.segment.length = unit(0, 'lines'),
    segment.size = 0.5,
    segment.alpha = 0.5,
    xlim  = c(2.2, NA),
  ) +
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  ggplot2::theme(strip.text = element_blank(),
                 axis.title=element_blank(),
  )

paired 

pe24 <- paired %>%
  dplyr::filter(pop == "CRISPR/CAS9" & trait == "LPCs") %>%
  ggplot2::ggplot(aes( x = genotype, y = signalZ, group = species, color = factor(sign))) +
  ggplot2::geom_point() + geom_line() +
  ggplot2::coord_cartesian( ylim = c(-0.5, 0.6), xlim = c(1,2.5))  +
  ggplot2::ylab("Signal (Z-score)") + ggplot2::xlab("Genotype")   +
  ggplot2::scale_color_manual(values= c("#df1b23","#1260ad") ) +
  ggplot2::scale_x_discrete(labels = c(expression(italic("wt")), expression(italic("CR")))) +
  # ggplot2::facet_wrap(pop ~ trait, scales = "free_y") +
  ggpubr::stat_pvalue_manual(
    wilcox_2 %>% dplyr::filter(trait == "LPCs"), label = "p = {pval_fmt}", bracket.nudge.y = 0.1,
  ) +
  ggrepel::geom_text_repel(
    data= paired  %>%
      dplyr::filter(pop == "CRISPR/CAS9" & species %in% c("LPC..18.1.","LPC..18.2.")),
    aes(label = label),
    hjust = 2,
    min.segment.length = unit(0, 'lines'),
    segment.size = 0.5,
    segment.alpha = 0.5,
    xlim  = c(2.2, NA)
  ) +
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  ggplot2::theme(strip.text = element_blank(),
                 axis.title=element_blank(),
  )


fig <- ggpubr::ggarrange(
  ggpubr::ggarrange(pe11,
                    pe12,
                    align = "hv", nrow = 2),
  ggpubr::ggarrange(pe21,pe22,
                    pe23, pe24,
                    align = "hv", nrow = 2, ncol =2),
  ncol = 2
  
) +
  theme(strip.text = element_blank(),
        axis.title.y=element_blank(),
        plot.margin = unit(c(0.05,0.075, 0.05, 0.05), "npc")
  )


line_x0 <- "Genotype"
line_x1 <- "PCs"
line_x2 <- "LPCs"
line_x3 <- "PCs/LPCs"

line_y0 <- "Z-score"
line_y1 <- "BILs"
line_y2 <- expression(italic("3@8.5"))
line_y3 <- "CRISPR/CAS9"
line_y4 <- expression(italic("ZmPLA1.2"))

fig3CD <- cowplot::ggdraw(fig) + 
 # cowplot::draw_label(line_x0, x = 0.45, y = 0.05) +
  cowplot::draw_label(line_x1, x = 0.15, y = 0.97) + # use relative coordinates for positioning
  cowplot::draw_label(line_x2, x = 0.285, y = 0.97) +
  cowplot::draw_label(line_x3, x = 0.42, y = 0.97) + # use relative coordinates for positioning
  cowplot::draw_label(line_x1, x = 0.6, y = 0.97) +
  cowplot::draw_label(line_x2, x = 0.817, y = 0.97) +
  cowplot::draw_label(line_y0, x = 0.05, y = 0.5, angle = 90) +
  cowplot::draw_label(line_y1, x = 0.95, y = 0.75, angle = -90) + # use relative coordinates for positioning
  cowplot::draw_label(line_y2, x = 0.93, y = 0.75, angle = -90) +
  cowplot::draw_label(line_y3, x = 0.95, y = 0.3, angle = -90) + # use relative coordinates for positioning
  cowplot::draw_label(line_y4, x = 0.93, y = 0.3, angle = -90)
  
pdf(file= "Figure3CD.pdf", height = 5, width = 12)
fig3CD
dev.off()
```



##F1 expression data

```{r}

exp_valuesPLALPCAT <- read_csv("data/fig_4/ZmPla-Lpcat.csv")

ZmPla1_exp <- exp_valuesPLALPCAT %>%
  filter(gene=="ZmPla1.2") %>%
  #filter(fertilizer_tx=="high") %>%
	ggplot(aes(x = temp_tx, y = expression, color = temp_tx)) +
  scale_color_manual(values=c("#1260ad","#df1b23", "#fd6820", "#14ade8")) +
	geom_jitter(position=position_jitter(0.1), size = 3) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange",
               color = "black") +
  #ylim(0,8) +
  labs(x = "Temperature", y = "*ZmPLA1.2*") +
  md_theme_classic() +
  #theme_cowplot(16) +
  #facet_grid(cols = vars(line), rows = vars(fertilizer_tx)) +
  facet_grid(cols = vars(line)) +
  #coord_fixed(ratio = 0.9) +
  theme(legend.position="none",
        strip.background = element_rect(colour="white", fill="white"))

ZmLpcat_exp <- exp_valuesPLALPCAT %>%
  filter(gene=="ZmLpcat") %>%
  #filter(fertilizer_tx=="high") %>%
	ggplot(aes(x = temp_tx, y = expression, color = temp_tx)) +
  scale_color_manual(values=c("#1260ad","#df1b23", "#fd6820", "#14ade8")) +
	geom_jitter(position=position_jitter(0.1), size = 3) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange",
               color = "black") +
  #ylim(0,8) +
  ylab("ZmLpcat1") +
  xlab("Temperature") +
  theme_cowplot(16) +
  #facet_grid(cols = vars(line), rows = vars(fertilizer_tx)) +
  facet_grid(cols = vars(line)) +
  #coord_fixed(ratio = 0.9) +
  theme(legend.position="none",
        strip.background = element_rect(colour="white", fill="white"))

ratio_exp <- exp_valuesPLALPCAT %>%
  filter(gene=="Ratio") %>%
  #filter(fertilizer_tx=="high") %>%
	ggplot(aes(x = temp_tx, y = expression, color = temp_tx)) +
  scale_color_manual(values=c("#1260ad","#df1b23", "#fd6820", "#14ade8")) +
	geom_jitter(position=position_jitter(0.1), size = 3) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange",
               color = "black") +
  #ylim(0,8) +
  ylab("ZmPla1.2/ZmLpcat1") +
  xlab("Temperature") +
  theme_cowplot(16) +
  #facet_grid(cols = vars(line), rows = vars(fertilizer_tx)) +
  facet_grid(cols = vars(line)) +
  #coord_fixed(ratio = 0.9) +
  theme(legend.position="none",
        strip.background = element_rect(colour="white", fill="white"))

plot_grid(ZmPla1_exp, ZmLpcat_exp, ratio_exp, ncol = 1)

```


## Fig 4D Recombination breakpoint + Lipid Phenotype overlay: log(PCs/PCs)

```{r}

library(qtl)
library(dplyr)
library(ggplot2)


data_dir <- "../data/fig_3" # all the source data is the same as Fig_3
geno_file  <- file.path(data_dir, "B73xPT_AGPv3_genotype.csv")
pheno_file <- file.path(data_dir, "B73xPT_combined_phenotype.csv")
base_pheno_file <- file.path(data_dir, "B73xPT_base_phenotype.csv")

cross <- read.cross(
  format = "csvs",  file = geno_file,
  na.strings = c("-",NA),
  phefile = pheno_file,
  genotypes = c("A","B")) %>%
  convert2riself() %>%
  jittermap() %>%
  calc.genoprob( step = 1)
cross$pheno$id <- gsub("LANMLR17B","B",cross$pheno$id)

  
S3_8542287 <-factor(pull.geno(cross,3)[, "S3_8542287"])
names(S3_8542287) <- gsub("LANMLR17B","B",cross$pheno$id)
levels(S3_8542287) <-  c("B73", "PT")

pheno <- cross$pheno  %>%
  dplyr::select(id, LPCs, PCs, PCs.LPCs) %>%
  dplyr::left_join(
    base_pheno %>%  
      dplyr::mutate( log10 = log10(PCs.LPCs)) %>%
      group_by(id) %>%
      dplyr::summarise(mean_log10 = mean(log10))
  ) %>%
  dplyr::mutate(S3_8542287 = S3_8542287[id]) %>%
  dplyr::filter(!is.na(S3_8542287) & !is.na(PCs.LPCs)) %>%
  dplyr::arrange(mean_log10) %>%
  dplyr::mutate(
    id = forcats::fct_reorder(id, mean_log10,.fun = identity)
    ) %>%
  dplyr::arrange(-mean_log10)

n <- 18
top_n <- pheno$id[1:n]


peak_idx <- which(colnames(cross$geno$`3`$data) == "S3_8542287")


bg_data <- cross$geno$`3`$data[,c((peak_idx-2):(peak_idx))] %>% 
  as.data.frame() %>%
  dplyr::mutate(id = factor(cross$pheno$id)) %>%
  dplyr::filter(id %in% as.character(pheno$id)) %>%
  dplyr::mutate(id = factor(id , levels = levels(pheno$id))) %>%
  tidyr::pivot_longer(cols = tidyselect::contains("_"),
                      names_to = "marker", values_to = "genotype") %>%
  tidyr::separate(marker, c("chr","pos"), sep = "_",remove  = FALSE) %>%
  dplyr::mutate(pos = as.integer(pos)) %>%
  dplyr::arrange(id,pos) %>%
  dplyr::group_by(id) %>%
  dplyr::mutate(len = pos - dplyr::lag(pos, default = pos[1])) %>%
  dplyr::mutate(lag_pos = dplyr::lag(pos)) %>%
  dplyr::mutate(cumlen = cumsum(len)) %>%
  dplyr::mutate(br = len/2 + dplyr::lag(cumlen)) %>%
  dplyr::filter(!is.na(lag_pos)) %>%
  dplyr::mutate(marker = factor(marker, levels = rev(marker[1:3]))) %>%
  dplyr::mutate(genotype = factor( c("B73", "PT")[genotype])) %>%
  dplyr::ungroup()

scatter_data <- bg_data %>%
  dplyr::inner_join(
    base_pheno[,c(1:4,which(colnames(base_pheno) == "PCs.LPCs"))] %>%
      dplyr::mutate(id = gsub("LANMLR17B","B",id)),
    by = "id",
  ) %>% 
  dplyr::mutate(log10_PCs.LPCs = log10(PCs.LPCs))  %>%
  dplyr::mutate(id = forcats::fct_reorder(id,log10_PCs.LPCs,.fun = mean, na.rm = TRUE))


png(file = "Fig_4D.png", width = 7, height = 7, units = "in", res = 200)
# quartz()
bg_data %>%
  dplyr::inner_join(pheno) %>%
  dplyr::filter(id %in% top_n) %>%
  ggplot2::ggplot() + 
  xlab("") +
  # Add background columns for illustrating recombination
  ggplot2::geom_col(aes(x = id, y = len, group = marker, fill= genotype),
                    color="white") +
  # Add points:
  #
  # ggplot2::geom_point(
  #   data = scatter_data %>% dplyr::filter(id %in% top),
  #   aes( y = log10_PCs.LPCs*9.1^6, x = id, shape = FIELD),
  #   size =2, color = "grey75") +
  #
  ggplot2::scale_fill_manual(values = c("#df1b23","#1260ad")) +
  ggplot2::stat_summary(data = scatter_data %>% dplyr::filter(id %in% top), 
                        aes(x = id, y = log10_PCs.LPCs*9.1^6),
                        fun.data=mean_sdl,fun.args = list(mult = 1), col ="white", size = 0.8) +
  ggplot2::scale_y_continuous( 
    name = "Marker Position",
    breaks = bg_data$br[1:2],
    labels = round(bg_data$pos[1:2] /1e6,2),
    # Add a second axis and specify its features
    sec.axis = sec_axis(~./1e6, name="log10(PCs/LPCs)")
  ) +
  ggplot2::coord_flip() +
  ggpubr::theme_pubr(legend = "none", base_size = 18)
dev.off()
```
