---
title: "Fig 2 Code"
output: html_document
---
```{r}
rm(list = ls())
library(tidyverse)
library(data.table)
library(cowplot)
library(wesanderson)
library(ggplot2)
library(MASS)
library(qtl)
library(lme4)
library(arm)
library(lsmeans)
library(pbkrtest)
library(cowplot)
library(viridis)
library(qtlcharts)
library(readr)
library(ggfortify)
library(ggmap)
library(maps)
library(qtlcharts)
library(Hmisc)
library(ggpubr)
```

```{r}
# Load all files

gbs <- read_csv("data/hpl_GBS_data.csv")
mapping_file <- read_csv("data/mapping_file.csv")
AccID <- read_csv("data/AccID.csv")
allLines_mapping <- read_csv("data/allLines_with_GID.csv")
load("data/fig_2/pcadapt_corrected.Rimage")


# All chromosomes figure

colm %>%
	#filter(CHR == 5) %>%
	#filter(BP > 194500000) %>%
	#filter(BP < 196500000) %>%
	#filter(P > 900) %>%
	#ggplot(aes(x = BP, P, colour = P)) +
	ggplot(aes(x = BP/1000000, -log10(P), colour = -log10(P))) +
	scale_color_viridis(option = "plasma", direction = -1) +
	geom_point(size = 0.3) +
	theme_cowplot(9) +
	facet_wrap( ~ CHR, ncol=5)
	#geom_segment(x = 195.495, xend = 195.505, y = 110, yend = 110, colour = "red", size = 6) +
	#xlim(5379695, 5395010) +
	#ylim(0, 130)

# LPCAT1 figure

colm %>%
	filter(CHR == 5) %>%
	filter(BP > 194500000) %>%
	filter(BP < 196500000) %>%
	#filter(-log10(P) > 70) %>%
	#ggplot(aes(x = BP, P, colour = P)) +
	ggplot(aes(x = BP/1000000, -log10(P), colour = -log10(P))) +
	scale_color_viridis(option = "plasma", direction = -1) +
	geom_point(size = 1) +
	theme_cowplot(16) +
	#facet_wrap( ~ CHR, ncol=5)
	geom_segment(x = 195.443, xend = 195.454, y = 110, yend = 110, colour = "red", size = 6) +
	#xlim(5379695, 5395010) +
	ylim(0, 130)

	# ZmPLA1.2 figure

	colm %>%
	filter(CHR < 3) %>%
	filter(BP > 7500000) %>%
	filter(BP < 9500000) %>%
	filter(-log10(P) > 5) %>%
	#ggplot(aes(x = BP, P, colour = P)) +
	ggplot(aes(x = BP/1000000, -log10(P), colour = -log10(P))) +
	scale_color_viridis(option = "plasma", direction = -1) +
	geom_point(size = 2) +
	theme_cowplot(12) +
	geom_segment(x = 8.541, xend = 8.543, y = 120, yend = 120, colour = "red", size = 6) +
	#xlim(5379695, 5395010) +
	ylim(0, 130)
```


```{r}
colm %>%
	filter(CHR == 6) %>%
	filter(BP > 0) %>%
	filter(BP < 15974565) %>%
	filter(-log10(P) > 35) %>%
	#ggplot(aes(x = BP, P, colour = P)) +
	ggplot(aes(x = BP/1000000, -log10(P), colour = -log10(P))) +
	scale_color_viridis(option = "plasma", direction = -1) +
	geom_point(size = 1) +
	theme_cowplot(9)
	#facet_wrap( ~ CHR, ncol=5)
	#geom_segment(x = 195.495, xend = 195.505, y = 110, yend = 110, colour = "red", size = 6) +
	#xlim(5379695, 5395010) +
	#ylim(0, 130)
```

## Elevation bin plots

```{r}

# Source data from:
#  http://germinate.seedsofdiscovery.org/maize/#accession-overview
# SeeD GWAS GbS Analysisa ccessions (4022)
accn_file <- file.path('data','fig_2','germinate_SEED_GWAS_GBS_4022.tab')
accn_info <- read.table(file = accn_file, header = TRUE, quote = "",sep ="\t")
nrow(accn_info)
SEED_GWAS_GID <- read.table(file = file.path('data','fig_2', "SEED_GWAS_GID.tab"),
                            header = TRUE, quote = "",sep ="\t")

# There are one or more GWAS seed identifiers per general identifier in the table

GWAS_ID <- read.table(file = file.path('data','fig_2',"GWAS_ID.tab"), quote = "",sep ="\t")
GWAS_ID$taxa <- paste(GWAS_ID$V1,GWAS_ID$V2,GWAS_ID$V3,GWAS_ID$V4, sep = ":")

colnames(GWAS_ID)[1] <-"sample_id"

GWAS_ID <- GWAS_ID %>%
  dplyr::select(sample_id,taxa) %>%
  dplyr::inner_join(SEED_GWAS_GID)

mex <- accn_info %>%
  dplyr::filter(!is.na(locations_longitude) & !is.na(locations_longitude)) %>%
  dplyr::filter(countries_country_code3 =="MEX") %>% # use the geolocator to retrieve this info
  dplyr::select(accn_GID  ="general_identifier",collnumb, lon ="locations_longitude",
                lat="locations_latitude",  ele="locations_elevation", country = "countries_country_code3")

mex$ele_bin <- cut(
  mex$ele, breaks = seq(0, 3100, by= 100),
  labels = seq(50, 3100, by= 100)) %>%
  as.character() %>%
  as.integer()

bins <- mex %>%
  dplyr::group_by(ele_bin) %>%
  dplyr::summarise(ele_count = length(accn_GID))

mex_filtered <- mex %>% dplyr::inner_join(bins) %>%
  dplyr::filter(ele_count>10)   # Filter low n bins


##########################################################################
# Extracted genotypes from Romero Navarro data with Tassel
# input as tassel table
genotype <- read.table(file.path('data','fig_2','S3_8542287.tab'), header = TRUE, sep ="\t") %>%
  # dplyr::inner_join(read.table(file.path('data','fig_2','S4_5387180.tab'), header = TRUE, sep ="\t")  %>%
  dplyr::inner_join(read.table(file.path('data','fig_2','S5_195455216.tab'), header = TRUE, sep ="\t"))

colnames(genotype) <- c("taxa","S3_8542287", "S5_195455216")


mex_info <- GWAS_ID %>%
  dplyr::inner_join(mex_filtered) %>%
  dplyr::inner_join(genotype) %>%
  dplyr::group_by(ele_bin) %>%
  dplyr::mutate(S3_8542287_fr   = (2*sum(S3_8542287   == "C") + sum(S3_8542287   == "S")) / (2*length(taxa))) %>%
  # dplyr::mutate(S4_5387180_fr   = (2*sum(S4_5387180   == "T") + sum(S4_5387180   == "Y")) / (2*length(taxa)))  %>%
  dplyr::mutate(S5_195455216_fr = (2*sum(S5_195455216 == "G") + sum(S5_195455216 == "R")) / (2*length(taxa)))%>%
  dplyr::arrange(ele_bin) %>%
  as.data.frame()

ele_fr  <- mex_info %>%
  ungroup() %>%
  dplyr::select(ele_bin, ends_with("fr")) %>%
  dplyr::rename(Elevation ="ele_bin")

pla1_plot <- ele_fr %>% ggpubr::ggscatter(  x = "Elevation", y = "S3_8542287_fr",
                               add = "loess",               # Add regression line
                               conf.int = TRUE,             # Add confidence interval
                               color = "#1260ad") +
  ggplot2::ggtitle("3@8.5 Mb") +
  ggplot2::ylab("Minor Allele Frequency") +
  ggplot2::scale_x_continuous(breaks = seq(0, 3000, 1000), limits = c(0, 3000)) +
  ggplot2::scale_y_continuous(breaks = seq(0, 1, .25), limits = c(0, 1.15)) +
  ggpubr::theme_classic2(base_size = 20)



lpcat_plot <- ele_fr %>%
  ggpubr::ggscatter( x = "Elevation",
                     y = "S5_195455216_fr",
                     add = "loess",               # Add regression line
                     conf.int = TRUE,  # Add confidence interval
                     color = "#1260ad") +
  ggplot2::ggtitle("5@195.5 Mb") +
  ggplot2::ylab("Minor Allele Frequency") +
  ggplot2::scale_x_continuous(breaks = seq(0, 3000, 1000), limits = c(0, 3000)) +
  ggplot2::scale_y_continuous(breaks = seq(0, 1, .25), limits = c(0, 1.15)) +
  ggpubr::theme_classic2(base_size = 20)

pdf(file.path('output',"fig_1CD.pdf"), width = 14)
ggpubr::ggarrange(pla1_plot,lpcat_plot) %>% print()
dev.off()
```
