---
title: "Fig 3 Code"
output: html_document
editor_options:
  chunk_output_type: inline
---

```{r}
rm(list = ls())
library(tidyverse)
library(data.table)
library(cowplot)
library(wesanderson)
library(ggplot2)
library(MASS) 
library(qtl)
library(lme4) 
library(arm) 
library(lsmeans) 
library(pbkrtest)
library(cowplot) 
library(viridis)
library(qtlcharts)
library(readr)
library(ggfortify)
library(ggmap)
library(maps)
library(qtlcharts)
library(Hmisc)
library(pegas)
library(mdthemes)
```



## Fig 3B Annotated QTL LOD profile

```{r}
library(dplyr)
library(qtl)
library(GenomicRanges)
library(ggbio)
library(igraph)
library(qtl)
data_dir <- "../data/fig_3"
source(file.path(data_dir,"detect_peaks.R"))

##### QTL analysis ##########################################################
n_threads <- 5
n_perm <- 1000
seed <- 1234567890
set.seed(seed)

geno_file  <- file.path(data_dir, "B73xPT_AGPv3_genotype.csv")
pheno_file <- file.path(data_dir, "B73xPT_combined_phenotype.csv")

cross <- read.cross(
  format = "csvs",  file = geno_file,
  na.strings = c("-",NA),
  phefile = pheno_file,
  genotypes = c("A","B")) %>%
  convert2riself() %>%
  jittermap() %>%
  calc.genoprob( step = 1)

# Select phenotypes with up to 50% missing data ###############################
# missing phenotypes will generate errors

nind <- nind(cross)
nphe <- nphe(cross)

missing <- colSums(is.na(cross$pheno))
missing_thresh <- 0.5

# this comes from the readcross warning
# RILs with genotype but no phenotype
pheno_col <- which( missing < missing_thresh * nind + 4) 
pheno_col <- pheno_col[-1]


single_scan <- scanone(cross, pheno.col = pheno_col, method = "hk")
perms <- scanone(cross, pheno.col = pheno_col,
                 n.perm = n_perm, method = "hk", n.cluster = n_threads)


peaks <- get_peak_table(perms = perms, single_scan = single_scan)

refined_peaks <- refine_peaks(peaks, perms = perms, single_scan = single_scan) %>%
  select(trait,lod,thresh_adj,thresh, everything()) %>%
  arrange(-lod) %>% tibble() %>%
  print(n = 150)

new_peaks  <- refined_peaks[refined_peaks$lod > refined_peaks$thresh_adj,]

# Variance explained by  3@8.5

hpc1_pos <- pull.map(cross = cross)[[3]]["S3_8542287"]

q1 <- makeqtl(sim.geno(cross), 3, hpc1_pos, "3@8.5")

qtraits <-  c("PCs","LPCs","PCs.LPCs")

peaks %>%
  dplyr::filter(trait %in% qtraits) %>%
  dplyr::select(trait, pos, marker, marker_lod)

trait_var <- lapply(qtraits, function(x){
  fit <- fitqtl(sim.geno(cross) , 
       pheno.col=which(colnames(cross$pheno) == x), 
       dropone = TRUE,
       q1)
  fit$result.full[1,5]
})

names(trait_var) <- qtraits
trait_var

base_size <- 24
       
p1 <- single_scan  %>%
  dplyr::select( chr,pos, LPCs, PCs,PCs.LPCs) %>%
  tidyr::pivot_longer(cols = c("LPCs", "PCs","PCs.LPCs"),
                      names_to = "trait",values_to = "LOD") %>%
  dplyr::mutate(trait = factor(trait, levels = c( "PCs","LPCs", "PCs.LPCs"))) %>%
  ggplot2::ggplot() +
  ggplot2::geom_line(aes(x = pos, y = LOD, col = trait))+
  ggplot2::xlab("Chromosome") +
  ggplot2::ylab("LOD") +
  ggplot2::geom_hline(aes(yintercept = LPCs_thresh), col = pal[1], lty =2)+
  ggplot2::facet_wrap(.~ as.factor(chr) ,
                      scales = "free_x",
                      strip.position = "bottom",
                      ncol = 10)  +
  ggplot2::geom_rug(aes(x = pos),sides="b", size =0.01) +
  ggpubr::theme_classic2(base_size = base_size) +
  ggplot2::theme(legend.position = "none",
                 strip.background = ggplot2::element_blank(),
                  strip.placement = "outside",
                  axis.text.x = ggplot2::element_text(size =14, angle = 90)
                 )


p2 <- single_scan  %>%
  dplyr::filter(chr==3) %>%
  dplyr::select( chr,pos, LPCs, PCs,PCs.LPCs) %>%
  tidyr::pivot_longer(cols = c("LPCs", "PCs","PCs.LPCs"),
                      names_to = "trait",values_to = "LOD") %>%
  dplyr::mutate(trait = factor(trait, levels = c( "PCs","LPCs", "PCs.LPCs"))) %>%
  ggplot2::ggplot() +
  ggplot2::geom_line(aes(x = pos, y = LOD, col = trait))+
  ggplot2::geom_hline(aes(yintercept = LPCs_thresh), col = pal[1], lty =2)+
  ggplot2::geom_vline(aes(xintercept = hpc1_pos), col = "red") +
  ggplot2::geom_rug(aes(x = pos),sides="b", size =0.1) +
  ggpubr::theme_classic2(base_size = base_size) +
  ggplot2::theme( plot.margin = unit(c(0,0,0,0), "lines"),
                  legend.position = "none",
                  axis.title = ggplot2::element_blank())

plot.with.inset <-
  cowplot::ggdraw() +
  cowplot::draw_plot(p1) +
  cowplot::draw_plot(p2, x = 0.31, y = .46, width = .68, height = .45)


base_size <- 18

# png(file = "Figure3A.png", height = 616, width = 1168)
# pdf(file = "Figure3A.pdf", height = 6.16, width = 11.68)

# quartz(height = 2*3.08, width = 2*5.84)

print(plot.with.inset)


grid::grid.text("LOD",
                just = "left", x = 0.3 , y = 0.95,  gp=grid::gpar(fontsize= base_size))
grid::grid.text("cM",
                just = "left", x = 0.95, y = 0.485,  gp=grid::gpar(fontsize= base_size))

grid::grid.text(expression(italic("HPC1")),
                just = "left", x = 0.48, y = 0.93, gp=grid::gpar(fontsize= base_size))
grid::grid.text(expression(italic("qPCs/LPCs3@8.5")),
                just = "left", x = 0.53, y = 0.85, gp=grid::gpar(fontsize= base_size))
grid::grid.text(expression(italic("qLPCs3@8.5")),
                just = "left",  x = 0.54, y = 0.71, gp=grid::gpar(fontsize= base_size))
grid::grid.text(expression(italic("qPCs3@8.5")),
                just = "left", x = 0.55, y = 0.66,  gp=grid::gpar(fontsize= base_size))

grid::grid.lines(x = unit(c(0.5188, 0.53), "npc"),
                 y = unit(c(0.89, 0.86), "npc"),
                 gp = grid::gpar(col= pal[3]))

grid::grid.lines(x = unit(c(0.513, 0.538), "npc"),
                 y = unit(c(0.679, 0.71), "npc"),
                 gp = grid::gpar(col= pal[2]))

grid::grid.lines(x = unit(c(0.513, 0.548), "npc"),
                 y = unit(c(0.627, 0.66), "npc"),
                 gp = grid::gpar(col= pal[1]))

grid::grid.lines(x = unit(c(0.272, 0.361), "npc"),
                 y = unit(c(0.288, 0.437), "npc"),
                 gp = grid::gpar(fill="black"),
                 arrow = grid::arrow(length = unit(0.1, "inches"),
                                     ends="last", type="closed"))
grid::grid.lines(x = unit(c(0.342, 0.95), "npc"),
                 y = unit(c(0.288, 0.435), "npc"),
                 gp = grid::gpar(fill="black"),
                 arrow = grid::arrow(length = unit(0.1, "inches"),
                                     ends="last", type="closed"))
dev.off()
 
```


## Fig 3E  F1 expression data

```{r}

exp_valuesPLALPCAT <- read.csv("data/fig_4/ZmPla-Lpcat.csv")
temp_order <- c("control", "cold")

exp_valuesPLALPCAT$temp_tx <- factor(exp_valuesPLALPCAT$temp_tx,levels = temp_order )

stat.test <- exp_valuesPLALPCAT %>%
  filter(gene=="ZmPla1.2") %>%
  group_by(line) %>%
  rstatix::t_test(expression ~ temp_tx, p.adjust.method = "BH") %>%
  rstatix::adjust_pvalue(method = "bonferroni") %>%
  rstatix::add_xy_position(x = "supp")
stat.test 



ZmPla1_exp <- exp_valuesPLALPCAT %>%
  filter(gene=="ZmPla1.2") %>%
  ggplot(aes(x = temp_tx, y = expression, color = temp_tx)) +
  scale_color_manual(values=c("#df1b23","#1260ad")) +
  geom_jitter(position=position_jitter(0.1), size = 3) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange",
               color = "black", size = 0.8 ) +
  ylim(c(1,8))+
  ggpubr::stat_pvalue_manual(stat.test, tip.length = 0.01, size = 6) +
  labs(x = "Temperature", y = "*HPC1* (log<sub>2</sub>[CPM])") +
  md_theme_classic(base_size = 25,) +
  facet_grid(cols = vars(line)) +
  theme(legend.position="none",
        strip.background = element_rect(colour="white", fill="white"))

quartz()
ZmPla1_exp

pdf(file = "Fig3E.pdf", height = 0.8*7.65, width = 0.9*8.22,)
print(ZmPla1_exp)
dev.off()
```


## Figure 3C Sums effects

```{r}
library(dplyr)
library(tidyr)
library(ggplot2)

base_size <- 13

# RILs data ------------------------------------------------------

library(qtl)
cross <- read.cross(
  format = "csvs",  file = file.path("B73xPT_AGPv3_genotype.csv"),
  na.strings = c("-",NA),
  phefile = file.path("B73xPT_combined_phenotype.csv"),
  genotypes = c("A","B")) %>%
  convert2riself() %>%
  jittermap() %>%
  calc.genoprob( step = 1)

pheno <- read.csv("B73xPT_combined_phenotype.csv") %>%
  dplyr::group_by(id)

cross$pheno$id

S3_8542287 <-factor(pull.geno(cross,3)[, "S3_8542287"])
levels(S3_8542287) <-  c("B73", "PT")

to_plot <- dplyr::inner_join(
  data.frame( id = cross$pheno$id,
              genotype = S3_8542287),
  pheno
)

plot_traits <- c("PCs","LPCs", "PCs.LPCs")
species_effect <-  to_plot %>% 
  dplyr::select(id, genotype, all_of(plot_traits)) %>%
  dplyr::mutate_if(is.numeric, scale) %>%
  tidyr::pivot_longer(all_of(plot_traits), names_to = "species", values_to = "Zscore" )
species_effect$Zscore <- species_effect$Zscore[,1]

sp_label <- c("PCs","LPCs", "PCs/LPCs")
names(sp_label) <- plot_traits
facet_labeller <- ggplot2::labeller(species = sp_label) 

y_pos <- species_effect  %>%
  dplyr::group_by(species) %>%
  rstatix::get_y_position(Zscore ~ genotype) %>%
  dplyr::pull(y.position)

t_RILs <- rbind(
  species_effect %>%
    dplyr::filter(species == "PCs")  %>%
    rstatix::t_test(Zscore ~ genotype) %>%
    rstatix::add_xy_position(x = "genotype"),
  species_effect %>%
    dplyr::filter(species == "LPCs")  %>%
    rstatix::t_test(Zscore ~ genotype) %>%
    rstatix::add_xy_position(x = "genotype"),
  species_effect %>%
    dplyr::filter(species == "PCs.LPCs") %>%
    rstatix::t_test(Zscore ~ genotype) %>%
    rstatix::add_xy_position(x = "genotype")
) %>%
  rstatix::adjust_pvalue(method = "BH") %>%
  rstatix::add_significance()  %>%
  dplyr::mutate( p_raw = p, p = p.adj,
                 pval_fmt = format.pval(p.adj, digits = 2),
                 species = factor(plot_traits, levels = plot_traits)) %>%
  dplyr::mutate(y.position = y_pos) %>%
  dplyr::select(species, everything())
as.data.frame(t_RILs)


pe11 <- species_effect %>%
  dplyr::filter(!is.na(genotype)) %>%
  dplyr::mutate(species = factor(species, levels = plot_traits)) %>%
  ggplot2::ggplot(aes(x = genotype, y = Zscore, color = genotype)) +
  ggplot2::geom_jitter(position=position_jitter(0.2), size = 1.5) +
  ggplot2::stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1),
                        geom="pointrange", color = "black", size = 0.5)+
  ggplot2::scale_y_continuous(expand = expansion(mult = c(0.1, 0.2))) +
  ggplot2::facet_wrap(. ~ species, scales = "free",
                      labeller = facet_labeller,
                      ncol = 3)  +
  ggpubr::stat_pvalue_manual(
    t_RILs, label = "{pval_fmt}",bracket.nudge.y = 0.1,
  ) +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad")) +
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  ggplot2::theme( axis.title = element_blank(),
                  strip.text = element_blank())


# CRISPR data -------------------------------------------------------------------

base_size <- 13

MS2020 <- read.csv("/Users/fvrodriguez/Projects/NCSU/05_PLA1/pla_phyt_20.csv") %>%
  dplyr::mutate(geno = gsub("hom", "mut", geno)) %>%
  dplyr::rename(sample = "sample_id",
                genotype = "geno") %>%
  janitor::clean_names(case = "all_caps")

colnames(MS2020)[1:3] <- colnames(MS2020)[1:3] %>% tolower()


colnames(MS2020) <- gsub("_([A-Z])$",".\\1",colnames(MS2020), perl =  TRUE)
with_isomers <- colnames(MS2020)[grepl("\\.([A-Z])$",colnames(MS2020))]

# Summarize over  desaturation isomers

isomer_sum <- MS2020 %>%
  dplyr::select(with_isomers) %>%
  t() %>% as.data.frame() %>%
  group_by(., id = gsub('\\..*', '', rownames(.))) %>%
  summarise_all(sum, na.rm = TRUE) %>%
  data.frame() %>%
  tibble::column_to_rownames(var = 'id') %>%
  t() 

MS2020 <- cbind(MS2020 %>% dplyr::select(-with_isomers, -colnames(isomer_sum)), isomer_sum) 
sp_cols <- colnames(MS2020  %>% dplyr::select(-(sample:genotype))) %>% sort()

MS2021 <- MS2020 %>% dplyr::select(sample:genotype, sp_cols) 


MS2020 <- MS2020 %>%
  dplyr::mutate(PCs = rowSums(select(., matches("^PC_")) , na.rm = TRUE),
                LPCs = rowSums(select(., matches("LPC_")) , na.rm = TRUE)) %>% 
  dplyr::mutate(PCs.LPCs = PCs/LPCs)


MS2021 <- read.csv("/Users/fvrodriguez/Desktop/2021-lipids.csv") %>%
  janitor::clean_names(case = "all_caps")

colnames(MS2021)[1:5] <- colnames(MS2021)[1:5] %>% tolower()



colnames(MS2021) <- gsub("_([A-Z])$",".\\1",colnames(MS2021), perl =  TRUE)
with_isomers <- colnames(MS2021)[grepl("\\.([A-Z])$",colnames(MS2021))]

# Summarize over  desaturation isomers

isomer_sum <- MS2021 %>%
  dplyr::select(with_isomers) %>%
  t() %>% as.data.frame() %>%
  group_by(., id = gsub('\\..*', '', rownames(.))) %>%
  summarise_all(sum, na.rm = TRUE) %>%
  data.frame() %>%
  tibble::column_to_rownames(var = 'id') %>%
  t() 

MS2021 <- cbind(MS2021 %>% dplyr::select(-with_isomers, -colnames(isomer_sum)), isomer_sum) 
sp_cols <- colnames(MS2021  %>% dplyr::select(-(site_year:sample))) %>% sort()

MS2021 <- MS2021 %>% dplyr::select(site_year:sample, sp_cols) 

MS2021 <- MS2021 %>%
  dplyr::mutate(PCs = rowSums(select(., matches("^PC_")) , na.rm = TRUE),
                LPCs = rowSums(select(., matches("LPC_")) , na.rm = TRUE)) %>% 
  dplyr::mutate(PCs.LPCs = PCs/LPCs)

shared2020 <- c("sample", "genotype",
                colnames(MS2021)[colnames(MS2021) %in% colnames(MS2020)][-1])

shared2021 <- c("sample", "group", 
                colnames(MS2021)[colnames(MS2021) %in% colnames(MS2020)][-1])

lipid2020 <- MS2020  %>% select(shared2020) %>%
  filter(genotype  %in% c("wt","mut")) %>%
  mutate(PCs.LPCs = PCs/LPCs)  %>%
  mutate(year = 2020)  %>%
  select(year, everything())

scaled2020 <- lipid2020 %>%
  dplyr::mutate_if(is.numeric, scale) %>%
  mutate(year = 2020)


lipid2021 <- MS2021 %>% select(c(group,shared2021))  %>%
  filter(grepl("phytotron",group))  %>%
  mutate(genotype = factor(group, levels = c("phytotron_wt","phytotron_CR"))) %>%
  mutate(genotype =  recode_factor( genotype, phytotron_wt = "wt",
                                    phytotron_CR = "mut")
  ) %>%
  select(shared2020) %>%
  mutate(PCs.LPCs = PCs/LPCs)  %>%
  mutate(year = 2021) %>%
  select(year, everything())

scaled2021 <- lipid2021 %>%
  dplyr::mutate_if(is.numeric, scale) %>%
  mutate(year = 2021)

plot_traits <-c("PCs","LPCs", "PCs.LPCs")

species_effect <- rbind(scaled2020,scaled2021) %>%
  mutate(genotype = factor(genotype, levels =  c("wt","mut"))) %>%
  dplyr::select(sample,genotype, all_of(plot_traits)) %>%
  dplyr::mutate_if(is.numeric, scale) %>%
  tidyr::pivot_longer(all_of(plot_traits), names_to = "species", values_to = "Zscore") %>%
  dplyr::filter(!is.na(genotype)) %>%
  dplyr::mutate(species = factor(species, levels = plot_traits))

species_effect$Zscore <- species_effect$Zscore[,1]

sp_label <- c("PCs","LPCs", "PC/LPC")
names(sp_label) <- plot_traits
facet_labeller <- ggplot2::labeller(species = sp_label)

y_pos <- species_effect  %>%
  dplyr::group_by(species) %>%
  rstatix::get_y_position(Zscore ~ genotype) %>%
  dplyr::pull(y.position) 

t_CRISPR <- rbind(
  species_effect %>%
    dplyr::filter(species == "PCs")  %>%
    rstatix::t_test(Zscore ~ genotype) %>%
    rstatix::add_xy_position(x = "genotype"),
  species_effect %>%
    dplyr::filter(species == "LPCs")  %>%
    rstatix::t_test(Zscore ~ genotype) %>%
    rstatix::add_xy_position(x = "genotype"),
  species_effect %>%
    dplyr::filter(species == "PCs.LPCs") %>%
    rstatix::t_test(Zscore ~ genotype) %>%
    rstatix::add_xy_position(x = "genotype")
) %>%
  rstatix::adjust_pvalue(method = "BH") %>%
  rstatix::add_significance()  %>%
  dplyr::mutate( p_raw = p, p = p.adj,
                 pval_fmt = format.pval(p.adj, digits = 2),
                 species = factor(plot_traits, levels = plot_traits)) %>%
  dplyr::mutate(y.position = y_pos) %>%
  dplyr::select(species, everything())
as.data.frame(t_CRISPR)

pe12 <- species_effect %>%
  dplyr::filter(!is.na(genotype)) %>%
  dplyr::mutate(species = factor(species, levels = plot_traits)) %>%
  ggplot2::ggplot(aes(x = genotype, y = Zscore, color = genotype)) +
  ggplot2::geom_jitter(position=position_jitter(0.2), size = 1.5) +
  ggplot2::stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1),
                        geom="pointrange", color = "black", size = 0.5) +
  ggplot2::scale_y_continuous(expand = expansion(mult = c(0.1, 0.1)))  +
  # crispr label
  ggplot2::scale_x_discrete(labels = c(expression(italic("hpc1"^"wt")),
                                       expression(italic("hpc1"^"Tins")))) +
  ggplot2::facet_wrap(. ~ species, scales = "free",
                      # labeller = facet_labeller,
                      ncol = 3)  +
  ggpubr::stat_pvalue_manual(
    t_CRISPR, label = "{pval_fmt}",bracket.nudge.y = 0.1,
  ) +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad")) +
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  ggplot2::theme( axis.title = element_blank(),
                  strip.text = element_blank())

# Output plot --------------------------------------------------
fig3C <- ggpubr::ggarrange(pe11,
                           pe12,
                           align = "hv", nrow = 2) +
  theme(strip.text = element_blank(),
        axis.title.y=element_blank(),
        plot.margin = unit(c(0.05,0.075, 0.05, 0.05), "npc")
  )

line_x1 <- "PCs"
line_x2 <- "LPCs"
line_x3 <- "PCs/LPCs"
line_y0 <- "Z-score"

pdf(file= "Figure3C.pdf", height = 5, width = 6)
cowplot::ggdraw(fig3C) +
  cowplot::draw_label(line_x1, x = 0.23, y = 0.97) + # use relative coordinates for positioning
  cowplot::draw_label(line_x2, x = 0.515, y = 0.97) +
  cowplot::draw_label(line_x3, x = 0.8, y = 0.97) + # use relative coordinates for positioning
  cowplot::draw_label(line_y0, x = 0.05, y = 0.5, angle = 90)
dev.off()

```

## Figure 3D Species effects

```{r}
library(dplyr)
library(tidyr)
library(ggplot2)

# Plot by species ---------------------------------------

## CRISPR data --------------------------------------------------------------
## I need to merge 2020 and 2021 results for phytotron

base_size <- 20

MS2020 <- read.csv("/Users/fvrodriguez/Projects/NCSU/05_PLA1/pla_phyt_20.csv") %>%
  dplyr::mutate(geno = gsub("hom", "mut", geno)) %>%
  dplyr::rename(sample = "sample_id",
                genotype = "geno") %>%
  janitor::clean_names(case = "all_caps") 

colnames(MS2020)[1:3] <- colnames(MS2020)[1:3] %>% tolower()
length(MS2020)


colnames(MS2020) <- gsub("_([A-Z])$",".\\1",colnames(MS2020), perl =  TRUE)
with_isomers <- colnames(MS2020)[grepl("\\.([A-Z])$",colnames(MS2020))]

# Summarize over  desaturation isomers

isomer_sum <- MS2020 %>%
  dplyr::select(with_isomers) %>%
  t() %>% as.data.frame() %>%
  dplyr::group_by(., id = gsub('\\..*', '', rownames(.))) %>%
  dplyr::summarise_all(sum, na.rm = TRUE) %>%
  data.frame() %>%
  tibble::column_to_rownames(var = 'id') %>%
  t() 

MS2020 <- cbind(MS2020 %>% dplyr::select(-with_isomers, -colnames(isomer_sum)), isomer_sum) 


sp_cols <- colnames(MS2020  %>% dplyr::select(-(sample:genotype))) %>% sort()

length(sp_cols)

MS2020 <- MS2020 %>% dplyr::select(sample:genotype, sp_cols) # %>%
          # dplyr::filter(sample != "B104")


MS2020 <- MS2020 %>%
  dplyr::mutate(PCs = rowSums(select(., matches("^PC_")) , na.rm = TRUE),
                LPCs = rowSums(select(., matches("LPC_")) , na.rm = TRUE)) %>% 
  dplyr::mutate(PCs.LPCs = PCs/LPCs)


MS2021 <- read.csv("/Users/fvrodriguez/Desktop/2021-lipids.csv") %>%
  janitor::clean_names(case = "all_caps")

colnames(MS2021)[1:5] <- colnames(MS2021)[1:5] %>% tolower()

colnames(MS2021) <- gsub("_([A-Z])$",".\\1",colnames(MS2021), perl =  TRUE)
with_isomers <- colnames(MS2021)[grepl("\\.([A-Z])$",colnames(MS2021))]


# Summarize over  desaturation isomers

isomer_sum <- MS2021 %>%
  dplyr::select(with_isomers) %>%
  t() %>% as.data.frame() %>%
  dplyr::group_by(., id = gsub('\\..*', '', rownames(.))) %>%
  dplyr::summarise_all(sum, na.rm = TRUE) %>%
  data.frame() %>%
  tibble::column_to_rownames(var = 'id') %>%
  t() 

MS2021 <- cbind(MS2021 %>% dplyr::select(-with_isomers, -colnames(isomer_sum)), isomer_sum) 
sp_cols <- colnames(MS2021  %>% dplyr::select(-(site_year:sample))) %>% sort()

MS2021 <- MS2021 %>% dplyr::select(site_year:sample, sp_cols) 

MS2021 <- MS2021 %>%
  dplyr::mutate(PCs = rowSums(select(., matches("^PC_")) , na.rm = TRUE),
                LPCs = rowSums(select(., matches("LPC_")) , na.rm = TRUE)) %>% 
  dplyr::mutate(PCs.LPCs = PCs/LPCs)

shared2020 <- c("sample", "genotype",
                colnames(MS2020)[colnames(MS2020) %in% colnames(MS2021)][-1])

shared2021 <- c("sample", "group", 
                colnames(MS2021)[colnames(MS2021) %in% colnames(MS2020)][-1])

length(shared2021)-5

lipid2020 <- MS2020  %>% select(shared2020) %>%
  filter(genotype  %in% c("wt","mut")) %>%
  mutate(PCs.LPCs = PCs/LPCs)  %>%
  mutate(year = 2020)  %>%
  select(year, everything())

scaled2020 <- lipid2020 %>%
  dplyr::mutate_if(is.numeric, scale) %>%
  mutate(year = 2020)

lipid2021 <- MS2021 %>% select(c(group,shared2021))  %>%
  filter(grepl("phytotron",group))  %>%
  mutate(genotype = factor(group, levels = c("phytotron_wt","phytotron_CR"))) %>%
  mutate(genotype =  recode_factor( genotype, phytotron_wt = "wt",
                                    phytotron_CR = "mut")
  ) %>%
  select(shared2020) %>%
  mutate(PCs.LPCs = PCs/LPCs)  %>%
  mutate(year = 2021) %>%
  select(year, everything())

scaled2021 <- lipid2021 %>%
  dplyr::mutate(sample = as.factor(sample)) %>%
  dplyr::mutate_if(is.numeric, scale) %>%
  mutate(year = 2021)

CRISPR_norm <- dplyr::left_join( 
  rbind(lipid2020, lipid2021) %>%
    dplyr::mutate(pop = "CRISPR/CAS9", sample = as.factor(sample)) %>%
    dplyr::mutate( genotype = factor(genotype, levels= c("wt","mut"))) %>%
    dplyr::select(pop,year:genotype, c(starts_with("PC_"),starts_with("LPC_"))) %>%
    tidyr::pivot_longer(
      cols = c(starts_with("PC_"),starts_with("LPC_")),
      names_to = "species", values_to = "signal"
    ) %>% 
    dplyr::mutate( signal = signal %>% as.vector()) %>%
    arrange(species,genotype,signal),
  
  rbind(scaled2020, scaled2021) %>%
    dplyr::mutate(pop = "CRISPR/CAS9", sample = as.factor(sample)) %>%
    dplyr::mutate( genotype = factor(genotype, levels= c("wt","mut"))) %>%
    dplyr::select(pop,year:genotype, c(starts_with("PC_"),starts_with("LPC_"))) %>%
    tidyr::pivot_longer(
      cols = c(starts_with("PC_"),starts_with("LPC_")),
      names_to = "species", values_to = "signalZ"
    ) %>% 
    dplyr::mutate( signalZ = signalZ %>% as.vector()) %>%
    arrange(species,genotype,signalZ)
) %>% 
  dplyr::mutate(species = sub("_", "",  species)) %>%
  dplyr::mutate(species = sub("_", ":",  species)) %>% 
  dplyr::mutate(species = sub("_", ":",  species)) %>% 
  dplyr::mutate(
    trait = case_when((grepl("^PC",species)) ~ "PCs",
                      (grepl("^LPC",species)) ~ "LPCs")) %>%
  dplyr::select(pop,year,trait,everything())

# RILs data ------------------------------------------------------

library(qtl)
cross <- read.cross(
  format = "csvs",  file = file.path("B73xPT_AGPv3_genotype.csv"),
  na.strings = c("-",NA),
  phefile = file.path("B73xPT_combined_phenotype.csv"),
  genotypes = c("A","B")) %>%
  convert2riself() %>%
  jittermap() %>%
  calc.genoprob( step = 1)

pheno <- read.csv("B73xPT_combined_phenotype.csv") %>%
  dplyr::group_by(id)

cross$pheno$id

S3_8542287 <-factor(pull.geno(cross,3)[, "S3_8542287"])
levels(S3_8542287) <-  c("B73", "PT")

RILs <- dplyr::inner_join(
  data.frame( id = cross$pheno$id,
              genotype = S3_8542287),
  pheno
)

RILs_norm <- NULL

RILs_norm <- rbind(
  RILs %>%
    dplyr::select(sample = id, genotype, starts_with("PC_") & !matches("LPC") & !matches("\\.")) %>%
    dplyr::filter(!is.na(genotype)) %>%
    tidyr::pivot_longer(cols = starts_with("PC_"),
                        names_to = "species", values_to = "signal"
    ) %>%
    dplyr::group_by(species) %>%
    dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                  trait = "PCs",
                  pop = "RILs")  %>%
    dplyr::ungroup(),
  
  RILs %>%
    dplyr::select(sample = id, genotype, starts_with("LPC_") & !matches("\\.")) %>%
    dplyr::filter(!is.na(genotype))  %>%
    tidyr::pivot_longer(
      cols = starts_with("LPC_"),
      names_to = "species", values_to = "signal"
    ) %>%
    dplyr::group_by(species) %>%
    dplyr::mutate(signalZ = scale(signal) %>% as.vector(),
                  # log10 = log10(signal) %>% as.vector(),
                  # log10Z = scale(log10(signal)) %>% as.vector(),
                  trait = "LPCs",
                  pop = "RILs")  %>%
    dplyr::ungroup()
) %>% 
  dplyr::mutate(species = sub("_", "",  species)) %>%
  dplyr::mutate(species = sub("_", ":",  species)) %>%               
  dplyr::mutate(pop = "RILs", year = 2018) %>%
  dplyr::select(pop,year,trait,everything())

exclude <- RILs_norm %>%
  dplyr::group_by(pop,species) %>%
  dplyr::summarise(na.sum = sum(is.na(signalZ))) %>% print(n = 60) %>%
  dplyr::filter(na.sum > 25) %>% pull(species)

RILs_norm <- RILs_norm %>%
  filter(! species %in% exclude)


# Make paired data set between CRISPR/CAS9 and RILs --------------------------
#check which species are shared 

paired <-  rbind(CRISPR_norm, RILs_norm) %>%
  dplyr::group_by(pop,trait,genotype,species) %>%
  dplyr::summarise(signal = mean(signal, na.rm =TRUE),
                   signalZ = mean(signalZ, na.rm =TRUE)) %>%
  dplyr::ungroup() %>%
  dplyr::group_by(pop,genotype,species) %>%
  dplyr::filter(!is.na(signal)) %>%
  dplyr::ungroup() %>%
  dplyr::group_by(pop,species) %>%
  dplyr::arrange(species,genotype) %>%
  dplyr::mutate( n = length(genotype)) %>%
  dplyr::filter( n == 2) %>% 
  dplyr::mutate( sign = sign(signalZ[2] - signalZ[1])) 

paired  %>% dplyr::arrange(pop,trait,species) %>% print(n=60)

sp_presence <- table(paired$genotype,paired$species)

#  these species are the ones going to the plot.
sp_shared <- colnames(sp_presence)[colSums(sp_presence) == 4]
sp_presence[,sp_shared]

#  Calculate difference per species --------------------------
#  for reordering the plot

CRISPR_diff <-  CRISPR_norm %>%
  dplyr::group_by(pop, trait, species,genotype) %>%
  dplyr::summarise( Zscore = mean(signalZ, na.rm =TRUE), err = sd(signalZ, na.rm =TRUE))


CRISPR_diff <- dplyr::left_join( 
  CRISPR_diff,
  CRISPR_diff %>% 
    dplyr::mutate(diff = Zscore -lag(Zscore)) %>%
    dplyr::filter(!is.na(diff)) %>%
    dplyr::select(-Zscore, -err, -genotype),
  by = c("pop","trait","species")
)  %>% 
  dplyr::filter(species %in% sp_shared) 

CRISPR_diff$species <-  forcats::fct_reorder(
  CRISPR_diff$species,
  -CRISPR_diff$diff
) 

CRISPR_PCs_norm <- CRISPR_norm %>% dplyr::filter(trait == "PCs")
CRISPR_LPCs_norm <- CRISPR_norm %>% dplyr::filter(trait == "LPCs")


SS_CIRSPR_PCs <- lapply( 
  split(CRISPR_PCs_norm , CRISPR_PCs_norm$species),
  FUN = function(x){
    rstatix::t_test(x, signal ~ genotype,alternative = "less") %>%
      rstatix::add_xy_position(x = "genotype")
  }
) %>% dplyr::bind_rows() %>%
  dplyr::mutate(species = levels(as.factor(CRISPR_PCs_norm$species))) %>%
  dplyr::arrange(p) %>%
  rstatix::adjust_pvalue(method = "BH") %>%
  rstatix::add_significance()  %>%
  dplyr::mutate( p_raw = p, p = p.adj,
                 pval_fmt = format.pval(p.adj, digits = 2)) 
colnames(SS_CIRSPR_PCs)

SS_CIRSPR_LPCs <- lapply( 
  split(CRISPR_LPCs_norm , CRISPR_LPCs_norm $species),
  FUN = function(x){
    rstatix::t_test(x, signal ~ genotype, alternative = "greater") %>%
      rstatix::add_xy_position(x = "genotype")
  }
) %>% dplyr::bind_rows() %>%
  dplyr::mutate(species = levels(as.factor(CRISPR_LPCs_norm$species))) %>%
  dplyr::arrange(p) %>%
  rstatix::adjust_pvalue(method = "BH") %>%
  rstatix::add_significance()  %>%
  dplyr::mutate( p_raw = p, p = p.adj,
                 pval_fmt = format.pval(p.adj, digits = 2)) 
colnames(SS_CIRSPR_PCs) == colnames(SS_CIRSPR_LPCs)


CRISPR_diff  <- CRISPR_diff %>%
  dplyr::left_join(rbind(SS_CIRSPR_PCs,SS_CIRSPR_LPCs), by = "species")

CRISPR_diff$p.adj.signif <- gsub("\\*+","\\*",CRISPR_diff$p.adj.signif)

CRISPR_diff$p.adj.signif[CRISPR_diff$genotype %in% c("wt", "B73")] <- ""


CRISPR_diff$species <-  forcats::fct_reorder(
  CRISPR_diff$species,
  CRISPR_diff$diff
) 



CRISPR_diff$species
p021 <- CRISPR_diff %>%
  # dplyr::mutate(species = factor(species, levels =sp_shared)) %>%
  dplyr::filter(trait == "PCs") %>%
  droplevels() %>%
  ggplot2::ggplot(aes(x = species, y = Zscore, group = genotype)) +
  ggplot2::geom_point( aes(color = genotype), size = 5) +
  ggplot2::geom_text( aes(x = species, y = max(Zscore)+0.1, label= p.adj.signif), size =6) +
  ggplot2::scale_color_manual(name = "",
                              labels = c(expression(italic("hpc1"^"wt")), 
                                         expression(italic("hpc1"^"Tins"))),
                              values = c("#df1b23","#1260ad")) +
  ggpubr::theme_pubr( base_size = base_size) +
  ggplot2::coord_flip() +
  theme(legend.position = "top",
        legend.text = element_text(size=20),
        legend.margin = margin(c(0,0,-20,0)),
        legend.box.margin = margin(c(0,0,0,0)),
        axis.title=element_blank())

p022 <- CRISPR_diff %>%
  dplyr::filter(trait == "LPCs") %>%
  droplevels() %>%
  ggplot2::ggplot(aes(x = species, y = Zscore, group = genotype)) +
  ggplot2::geom_point( aes(color = genotype), size = 5) +
  ggplot2::geom_text( aes(x = species, y = max(Zscore)+0.15, label= p.adj.signif), size =6) +
  ggplot2::scale_color_manual(values = c("#df1b23","#1260ad"))  +
  ggplot2::ylab("Z-score") + 
  ggplot2::coord_flip() + 
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  theme(# axis.text.x = element_text(angle = 90, vjust = 0.5),
    axis.title.y = element_blank())



pp2 <- ggpubr::ggarrange(p021, p022, nrow = 2, align = "hv", heights  = c(1.8,1))



RILs_PCs_norm <- RILs_norm %>% dplyr::filter(trait == "PCs")
RILs_LPCs_norm <-RILs_norm %>% dplyr::filter(trait == "LPCs")



SS_RILs_PCs <- lapply( 
  split(RILs_PCs_norm, RILs_PCs_norm$species),
  FUN = function(x){
    rstatix::t_test(x, signalZ ~ genotype) %>%
      rstatix::add_xy_position(x = "genotype")
  }
) %>% dplyr::bind_rows() %>%
  dplyr::mutate(species = levels(as.factor(RILs_PCs_norm$species))) %>%
  dplyr::arrange(p)  %>%
  rstatix::adjust_pvalue(method = "BH") %>%
  rstatix::add_significance()  %>%
  dplyr::mutate( p_raw = p, p = p.adj,
                 pval_fmt = format.pval(p.adj, digits = 2)) 

SS_RILs_PCs$p.adj.signif

SS_RILs_LPCs <- lapply( 
  split(RILs_LPCs_norm, RILs_LPCs_norm$species),
  FUN = function(x){
    rstatix::t_test(x, signalZ ~ genotype) %>%
      rstatix::add_xy_position(x = "genotype")
  }
) %>% dplyr::bind_rows() %>%
  dplyr::mutate(species = levels(as.factor(RILs_LPCs_norm$species))) %>%
  dplyr::arrange(p)  %>%
  rstatix::adjust_pvalue(method = "BH") %>%
  rstatix::add_significance()  %>%
  dplyr::mutate( p_raw = p, p = p.adj,
                 pval_fmt = format.pval(p.adj, digits = 2)) 


RILs_diff <-  RILs_norm %>%
  dplyr::group_by(pop, trait, species,genotype) %>%
  dplyr::summarise( Zscore = mean(signalZ, na.rm =TRUE))

RILs_diff <- left_join( 
  RILs_diff,
  RILs_diff %>% 
    dplyr::mutate(diff = Zscore -lag(Zscore)) %>%
    dplyr::filter(!is.na(diff)) %>%
    dplyr::select(-Zscore, -genotype),
  by = c("pop","trait", "species")
)   %>% 
  dplyr::filter(species %in% sp_shared) 

RILs_diff$species <-  forcats::fct_reorder(
  RILs_diff$species,
  -RILs_diff$diff
) 


sp_shared <- levels(RILs_diff$species)

RILs_diff  <- RILs_diff %>%
dplyr::left_join(rbind(SS_RILs_PCs,SS_RILs_LPCs), by = "species")


RILs_diff$p.adj.signif <- gsub("\\*+","\\*",RILs_diff$p.adj.signif)

RILs_diff$p.adj.signif[RILs_diff$genotype %in% c("wt", "B73")] <- ""

RILs_diff %>%
  dplyr::filter(trait == "PCs") %>%
  droplevels() %>%
  dplyr::arrange(-diff)

RILs_diff$species <-  forcats::fct_reorder(
  RILs_diff$species,
  RILs_diff$diff
) 


p011 <- RILs_diff %>%
  dplyr::filter(trait == "PCs") %>%
  droplevels() %>%
  ggplot2::ggplot(aes(x = species, y = Zscore, group = genotype)) +
  ggplot2::geom_point( aes(color = genotype), size = 5) +
  ggplot2::geom_text( aes(x = species, y = max(Zscore)+0.2, label= p.adj.signif), size =6) +
  ggplot2::coord_flip() +
  ggplot2::scale_color_manual(name = "",
                              labels = c(expression(italic("3@8.5"^"B73")), 
                                         expression(italic("3@8.5"^"PT"))),
                              values = c("#df1b23","#1260ad")) +
  ggpubr::theme_pubr(base_size = base_size) +
  theme(legend.position = "top",
        legend.text = element_text(size=20),
        legend.margin = margin(c(0,0,-20,0)),
        legend.box.margin = margin(c(0,0,0,0)),
        axis.title=element_blank())


p012 <- RILs_diff %>%
  dplyr::filter(trait == "LPCs") %>%
  droplevels() %>%
  ggplot2::ggplot(aes(x = species, y = Zscore, group = genotype)) +
  ggplot2::geom_point( aes(color = genotype), size = 5) +
  ggplot2::geom_text( aes(x = species, y = max(Zscore) +0.2, label= p.adj.signif), size =6) +
  ggplot2::scale_color_manual( values = c("#df1b23","#1260ad")) +
  ggplot2::ylab("Z-score") + 
  ggplot2::coord_flip() + 
  ggpubr::theme_pubr(legend ="none", base_size = base_size) +
  theme(# axis.text.x = element_text(angle = 90, vjust = 0.5),
        axis.title.y = element_blank())

pp1 <- ggpubr::ggarrange(p011, p012, nrow = 2, heights = c(1.8,1), align = "hv")

fig3D <- ggpubr::ggarrange(pp1, pp2, ncol =2, align = "hv", widths = c(1.2, 1))

pdf(file= "Figure3D.pdf", height = 8.08, width = 9.25)
fig3D
dev.off()

```



## Fig 3F Recombination breakpoint + Lipid Phenotype overlay: log(PCs/PCs)

```{r}

library(qtl)
library(dplyr)
library(ggplot2)

geno_file  <- file.path(data_dir, "B73xPT_AGPv3_genotype.csv")
pheno_file <- file.path(data_dir, "B73xPT_combined_phenotype.csv")
base_pheno_file <- file.path(data_dir, "B73xPT_base_phenotype.csv")

cross <- read.cross(
  format = "csvs",  file = geno_file,
  na.strings = c("-",NA),
  phefile = pheno_file,
  genotypes = c("A","B")) %>%
  convert2riself() %>%
  jittermap() %>%
  calc.genoprob( step = 1)
cross$pheno$id <- gsub("LANMLR17B","B",cross$pheno$id)

  
S3_8542287 <-factor(pull.geno(cross,3)[, "S3_8542287"])
names(S3_8542287) <- gsub("LANMLR17B","B",cross$pheno$id)
levels(S3_8542287) <-  c("B73", "PT")

pheno <- cross$pheno  %>%
  dplyr::select(id, LPCs, PCs, PCs.LPCs) %>%
  dplyr::left_join(
    base_pheno %>%  
      dplyr::mutate( log10 = log10(PCs.LPCs)) %>%
      group_by(id) %>%
      dplyr::summarise(mean_log10 = mean(log10))
  ) %>%
  dplyr::mutate(S3_8542287 = S3_8542287[id]) %>%
  dplyr::filter(!is.na(S3_8542287) & !is.na(PCs.LPCs)) %>%
  dplyr::arrange(mean_log10) %>%
  dplyr::mutate(
    id = forcats::fct_reorder(id, mean_log10,.fun = identity)
    ) %>%
  dplyr::arrange(-mean_log10)

n <- 18
top_n <- pheno$id[1:n]


peak_idx <- which(colnames(cross$geno$`3`$data) == "S3_8542287")


bg_data <- cross$geno$`3`$data[,c((peak_idx-2):(peak_idx))] %>% 
  as.data.frame() %>%
  dplyr::mutate(id = factor(cross$pheno$id)) %>%
  dplyr::filter(id %in% as.character(pheno$id)) %>%
  dplyr::mutate(id = factor(id , levels = levels(pheno$id))) %>%
  tidyr::pivot_longer(cols = tidyselect::contains("_"),
                      names_to = "marker", values_to = "genotype") %>%
  tidyr::separate(marker, c("chr","pos"), sep = "_",remove  = FALSE) %>%
  dplyr::mutate(pos = as.integer(pos)) %>%
  dplyr::arrange(id,pos) %>%
  dplyr::group_by(id) %>%
  dplyr::mutate(len = pos - dplyr::lag(pos, default = pos[1])) %>%
  dplyr::mutate(lag_pos = dplyr::lag(pos)) %>%
  dplyr::mutate(cumlen = cumsum(len)) %>%
  dplyr::mutate(br = len/2 + dplyr::lag(cumlen)) %>%
  dplyr::filter(!is.na(lag_pos)) %>%
  dplyr::mutate(marker = factor(marker, levels = rev(marker[1:3]))) %>%
  dplyr::mutate(genotype = factor( c("B73", "PT")[genotype])) %>%
  dplyr::ungroup()

scatter_data <- bg_data %>%
  dplyr::inner_join(
    base_pheno[,c(1:4,which(colnames(base_pheno) == "PCs.LPCs"))] %>%
      dplyr::mutate(id = gsub("LANMLR17B","B",id)),
    by = "id",
  ) %>% 
  dplyr::mutate(log10_PCs.LPCs = log10(PCs.LPCs))  %>%
  dplyr::mutate(id = forcats::fct_reorder(id,log10_PCs.LPCs,.fun = mean, na.rm = TRUE))


# png(file = "Fig_4D.png", width = 7, height = 7, units = "in", res = 200)
# quartz()
bg_data %>%
  dplyr::inner_join(pheno) %>%
  dplyr::filter(id %in% top_n) %>%
  ggplot2::ggplot() + 
  xlab("") +
  # Add background columns for illustrating recombination
  ggplot2::geom_col(aes(x = id, y = len, group = marker, fill= genotype),
                    color="white") +
  # Add points:
  #
  # ggplot2::geom_point(
  #   data = scatter_data %>% dplyr::filter(id %in% top),
  #   aes( y = log10_PCs.LPCs*9.1^6, x = id, shape = FIELD),
  #   size =2, color = "grey75") +
  #
  ggplot2::scale_fill_manual(values = c("#df1b23","#1260ad")) +
  ggplot2::stat_summary(data = scatter_data %>% dplyr::filter(id %in% top), 
                        aes(x = id, y = log10_PCs.LPCs*9.1^6),
                        fun.data=mean_sdl,fun.args = list(mult = 1), col ="white", size = 0.8) +
  ggplot2::scale_y_continuous( 
    name = "Marker Position",
    breaks = bg_data$br[1:2],
    labels = round(bg_data$pos[1:2] /1e6,2),
    # Add a second axis and specify its features
    sec.axis = sec_axis(~./1e6, name="log10(PCs/LPCs)")
  ) +
  ggplot2::coord_flip() +
  ggpubr::theme_pubr(legend = "none", base_size = 18)
# dev.off()
```
