By the same logic, we should change 'day' and 'night' to be more inclusive.---
title: "Nucleotide diversity of ZmPla1.2 across cds and promoter regions"
author: "Andi Kur"
date: "11/12/2020"
output: html_document
---

A script for estimation pi across the gene regions of PLA in maize

```{r setup}
#clear the R environment and load libraries
rm(list = ls())
library(tidyverse)
library(PopGenome)

```

Start by reading in FASTA file with all samples in single file
Note that the FASTA must be in its own directory. For estimating pi separately 
across coding and promoter regions, I put my subset FASTA into two separate folders:
  ./promoter/all_PLA_promoter.fasta
  ./cds/ALL/all_PLA_cds.fasta

````{r data}
#Use Popgenome to read in data for promoter and cds
  #must use include.unknown=FALSE so that uncertain nucleotides do not bias the pi estimate

#Promoter:
PLA_promoter <- readData("./promoter/", format = "FASTA", include.unknown = FALSE)
get.sum.data(PLA_promoter)

#cds:
PLA_cds <- readData("./cds/ALL/", format = "FASTA", include.unknown = FALSE)
get.sum.data(PLA_cds)

```

Because all of our samples are in a single FASTA file, we need to sort them by population.
To do this, write a tab delimited text file with 2 columns, with sample IDs in the
first column and population ID in the second.

``` {r labels}
#read population labels in
PLA_info <- read_delim("./PLApopulations.txt", delim = "\t")
PLA_info

#get data for the populations
populations <- split(PLA_info$ind, PLA_info$pop)

#now set for both coding and promoter objects
PLA_promoter <- set.populations(PLA_promoter, populations)
PLA_cds <- set.populations(PLA_cds, populations)

#check that it worked
PLA_promoter@populations
PLA_cds@populations
```

Next, calculate pi using PopGenome and the nucleotide.diversity.within method.
This will calculate pi for each sight in the cds and promoter regions within each
defined population. The output will then need to be transposed so that SNPs are in each 
row and populations are in the columns.


```{r pi}
#calculate nucleotide diversity
PLA_cds <- diversity.stats(PLA_cds, pi=TRUE)
PLA_promoter <- diversity.stats(PLA_promoter, pi=TRUE)

  #Extract nucleotide data for each of the populations from the GENOME object
  #use t to transpose the data, so that each row is a SNP and each column a pop
PLA_promoter_pi <- t(PLA_promoter@region.stats@nuc.diversity.within[[1]])
PLA_cds_pi <- t(PLA_cds@region.stats@nuc.diversity.within[[1]]) 


#need to get data into a more suitable format for visualization
# get SNP positions from the row names
position_promoter <- as.numeric(rownames(PLA_promoter_pi))
position_cds <- as.numeric(rownames(PLA_cds_pi))

#rename the matrix columns after the populations
colnames(PLA_promoter_pi) <- c("MexH", "MexL", "SAH", "SAL")
colnames(PLA_cds_pi) <- c("MexH", "MexL", "SAH", "SAL")
#combine into a data.frame and remove the row.names
PLA_promoter_nd <- data.frame(position, PLA_promoter_pi, row.names = NULL)
PLA_cds_nd <- data.frame(position, PLA_cds_pi, row.names = NULL)

PLA_promoter_nd <- as.tibble(PLA_promoter_nd)
PLA_cds_nd <- as.tibble(PLA_cds_nd)

PLA_promoter_nd
PLA_cds_nd

#Save
write.csv(PLA_promoter_nd, "~/Desktop/PLA_promoter.csv")
write.csv(PLA_cds_nd, "~/Desktop/PLA_cds.csv")
```


