---
title: "Supp Fig8 Code"
output: html_document
---

```
# Make phenotype table #########################################################
# Summer - winter data Hung et al 2012

# https://datacommons.cyverse.org/browse/iplant/home/shared/panzea/phenotypes/Hung_etal_2012_PNAS_data-120523.zip
gddtf <- read.csv("Dataset_S2.Photoperiod_paper_data_supplement_line_BLUPs.csv") %>%
  dplyr::filter(pop==27) %>%
  dplyr::select( Taxa = geno_code, gdd_anth_long, gdd_anth_short,gdd_anth_photo_resp
                 , gdd_silk_long, gdd_silk_short, gdd_silk_photo_resp)

# Kremling et al 2018 expression data.
# https://data.cyverse.org/dav-anon/iplant/commons/cyverse_curated/Kremling_Nature3RNASeq282_March2018/Expression_matrix/TASSEL_fmt_expression_w_covariates/

tissue_file <- list.files(pattern = "TASSEL.*Dups.txt")
tissue <-  gsub(".*and_Altnames_","",tissue_file)
tissue <-  gsub("_.*","",tissue)
names(tissue_file) <- tissue
tissue  <- c(tissue[c(1,2,4,5,3)], NA,tissue[c(6,7)])

tissue_file
#expression data
tissue_cor <- list()

# Specific genes for pair wise correlation plots of gene expression
exp_set <-  c("GRMZM2G353444","GRMZM2G481755", "GRMZM2G179264", "GRMZM2G171650")
names(exp_set) <- c("HPC1","LPCAT1", "ZCN8","MADS69")

exp_cor <- list()

for (t in tissue){
  if(is.na(t)){
    plot.new()
    next
  }

  RNA <-read.table(file=tissue_file[t], skip =2, header = TRUE)
  RNA$Taxa <- gsub("282set_","",RNA$Taxa)

  DTF_RNA <- gddtf %>%
    dplyr::inner_join(RNA) %>%
    dplyr::select(Taxa, gdd_anth_long, gdd_anth_short, gdd_anth_photo_resp, gdd_silk_long, gdd_silk_short, gdd_silk_photo_resp,everything())

  colnames(DTF_RNA) <- gsub("gdd_","",colnames(DTF_RNA))
  #DTF_RNA[,36:40]
  #DTF_RNA[,1:7]

  exp_cor[[t]] <-  DTF_RNA %>%
    dplyr::select(tidyselect::all_of( exp_set)) %>%
    dplyr::mutate(tissue = t)

  gpcor <- cor(DTF_RNA[,2:7], DTF_RNA[,38:ncol(DTF_RNA)], method="pearson")

  tissue_cor[[t]] <-  gpcor %>%  t()
}


simplified <- c("GRMZM2G353444","GRMZM2G481755", "GRMZM2G179264",
                "GRMZM2G171650","GRMZM2G700665","GRMZM2G154366",
                "GRMZM2G032339")
names(simplified) <- c("HPC1","HPC1","ZCN8","MADS69","RAP2.7","PAP",  "ZMM4")
simplified <- simplified[1:5]  # exclude PAP and ZMM4

check_all_tissues(cor_list= tissue_cor[aerial],  genes = simplified)

col_fun = colorRamp2(c(-0.5, 0, 0.5), c("blue", "white", "red"))



ht_list <- NULL  ## Heatmap(...) + NULL gives you a HeatmapList object
m <-NULL
tissue_cor[[t]][simplified,]
for(t in aerial) {

  m <- tissue_cor[[t]][simplified,]
row_names <- row.names(m)
row.names(m) <- names(simplified)
row.names(m)
ht_list = ht_list +
  ComplexHeatmap::Heatmap(
    m,
    cluster_columns = FALSE,
    cluster_rows = TRUE,
    column_title = t,
    col = col_fun)
}


M <- NULL
for(t in aerial) {
  m <- tissue_cor[[t]][simplified,]
  M <- cbind(M,m)
}

rownames(M) <- names(simplified)
colnames(M) <-rep(c("Anthesis Long","Anthesis Short","Anthesis Response",
      "Silking Long","Silking Short","Silking Response"),5)

col_fun = colorRamp2(c(-0.5, 0, 0.5), c("blue", "white", "red"))

ha <-  rowAnnotation(foo = anno_text(rownames(M)))

quartz(height = 2*2.30, width = 2*4.72)
hm <- ComplexHeatmap::Heatmap(M,
                              cluster_columns = FALSE,
                              cluster_rows = FALSE,
                              col = col_fun,
                              column_split = rep(aerial, each = 6),
                              heatmap_legend_param = list(title = "Correlation"),
                              # layer_fun = function(j, i, x, y, w, h, fill) {
                              #  ind_mat = restore_matrix(j, i, x, y)
                              #  grid.points(x[1:2], y[1:2], pch = 16, size = unit(1, "mm"))}
) + ha

pdf("Supp_Fig8.pdf, "height = 2*2.30, width = 2*4.72)
ComplexHeatmap::draw(hm,merge_legend = TRUE),
dev.off()

```