---
title: "supplementary_fig_5"
output: pdf_document
---
```{r}
library(ggplot2)
library(tidyr)
library(dplyr)
library(magrittr)
library(readr)
library(cowplot)
```
load dataset 1 from citep("doi:10.3835/plantgenome2015.04.0025") and list of PC pathway genes
```{r}
tissue_exp <- read.delim("data/Supp_fig5/Dataset_S1.txt", stringsAsFactors=FALSE) 
phospholipases <- read_csv("data/Supp_fig5/LIPAS-PWY.csv") # it can be dowloaded from Plant Metabolic Network
genes_in_qtl <-read_csv("data/Supp_fig5/genes_in_QTL.csv")
# make tissue_exp long
tissue_exp_long <- tissue_exp %>%
    gather(tissue, expression_val, -gene_name, -chromosome, -position_left, -position_right) 
    
# join phospholipases and tissue expression datasets
phospholipases_exp <- phospholipases %>%
    inner_join(tissue_exp_long, by = "gene_name") 

# create dataframe with only HPC1
tissue_exp_PLA <- phospholipases_exp %>%
    filter(expression_val > 100) %>%
    filter(gene_name == "GRMZM2G353444") 
```
Panel A
```{r}
phospholipases_exp %>%
  filter(expression_val > 100) %>% # to reduce the number of tissues plotted
    filter(expression_val < 6000) %>%
  ggplot(aes(x = reorder(tissue, expression_val), y = expression_val)) +
  geom_point(aes(alpha = 0.6)) +
    geom_point(data=tissue_exp_PLA, colour = "blue") +
    theme_cowplot(12) +
  coord_flip() +
    ylab("FPKM") +
    xlab("tissue")
```
Panel B
```{r}
tissue_exp_long  %>%
    #filter(sec_tissue == "Leaf") %>%
    filter(chromosome == "chr3") %>%
    filter(position_left > 7900000) %>%
    filter(position_left < 10000000)  %>%
    ggplot(aes(x = position_left/1000000, y = expression_val)) +
    geom_point(alpha = 0.8) +
    geom_point(data=tissue_exp_PLA, colour = "blue") +
    #geom_vline(xintercept = 7900000, colour="red") +
    #geom_vline(xintercept = 10000000, colour="red") +
    #xlim(7000000, 11000000) +
    theme_cowplot(16) +
    xlab("Mb") +
    ylab("FPKM") 
```
Panel C
```{r}
inbred_ase <- read_csv("data/Supp_fig5/waters_inbred_ase.csv") 
inbred_ase_tidy <- inbred_ase %>%
  gather(gene, expression_value, -Gene_ID, -chr, -start) %>%
  separate(gene, c("genotype", "treatment", "rep", "batch"), "_", extra = "drop") # tidy data
GRMZM2G353444 <- inbred_ase_tidy %>%
  filter(Gene_ID == "GRMZM2G353444") %>%
  filter(treatment == "cold" | treatment == "control"| treatment == "heat") %>%
  filter(genotype == "B73" | genotype == "Mo17" | genotype == "Oh43" | genotype == "PH207") %>%
  ggplot(aes(x = treatment,  y = expression_value, color = treatment)) +
  geom_jitter(position=position_jitter(0.1), size = 3) +
  #geom_jitter(width = 0.25, size = 2) +
  scale_color_manual(values=c("#1260ad", "#087f2a", "#df1b23")) +
  stat_summary(fun.data=mean_sdl,fun.args = list(mult = 1), geom="pointrange", color = "black") +
  #ggtitle("ZmPla1.2") +
  #theme(plot.title = element_text(size = 10)) +
  theme_cowplot(14) +
  theme(legend.position="none") +
  ylab("FPKM")
```